{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Records Found: 186\n",
      "50 Id's for PubMed IDs of articles related to plastic and enzymes\n",
      " ['33006156', '32758514', '32160855', '32153083', '32117139', '32115828', '31910206', '31870825', '31830656', '31800321', '31787276', '31740410', '31733968', '31632361', '31514776', '31469935', '31382051', '31324632', '31294717', '31280392', '31266305', '31247237', '31211775', '31026589', '30982949', '30979881', '30832692', '30815712', '30803567', '30761732', '30714483', '30639217', '30592151', '30471390', '30456640', '30430764', '30352276', '30304066', '30255674', '30133489', '30126174', '30054060', '29792416', '29666242', '29594338', '29590588', '29374183', '29359624', '29343717', '29188330']\n",
      "title: Draft genome sequence of plastic degrading Bacillus sp. AIIW2 isolated from the Arabian ocean.\n",
      "authors: ['Kumari A', 'Bano N', 'Chaudhary DR', 'Jha B']\n",
      "source: The endemic spread of plastic in the environment requires urgent need of a sustainable approach. Marine microbes found to have vast bioactivity and play a central role in biogeochemical cycling in the ocean; however, very few of them had been explored for biochemical cycling or plastic degradation. In the present study, we report the draft genome sequence of marine Bacillus sp. AIIW2 which was found to utilize plastic as a carbon source. The Bacillus sonorensis SRCM101395 was used as a reference genome for mapping the reads. The genome size of strain AIIW2 was approximately 4.4 Mb and composed of 4737 coding sequences with 45.7% G + C contents. The whole genome comparison of strain AIIW2 with three closest Bacillus strains showed strain specificity, the 16S ribosomal RNA sequence shows 99.93% similarity with Bacillus paralicheniformis KJ-16(T) (KY694465). This genome data would provide the genetic basis in developing plastic bioremediation approaches and discover the enzymes pertinent in the biodegradation processes.\n",
      "\n",
      "title: Bacterial and enzymatic degradation of poly(cis-1,4-isoprene) rubber: Novel biotechnological applications.\n",
      "authors: ['Andler R']\n",
      "source: Poly(cis-1,4-isoprene) rubber is a highly demanded elastomeric material mainly used for the manufacturing of tires. The end-cycle of rubber-made products is creating serious environmental concern and, therefore, different recycling processes have been proposed. However, the current physical-chemical processes include the use of hazardous chemical solvents, large amounts of energy, and possibly generations of unhealthy micro-plastics. Under this scenario, eco-friendly alternatives are needed and biotechnological rubber treatments are demonstrating huge potential. The cleavage mechanisms and the biochemical pathways for the uptake of poly(cis-1,4-isoprene) rubber have been extensively reported. Likewise, novel bacterial strains able to degrade the polymer have been studied and the involved structural and functional enzymes have been analyzed. Considering the fundamentals, biotechnological approaches have been proposed considering process optimization, cost-effective methods and larger-scale experiments in the search for practical and realistic applications. In this work, the latest research in the rubber biodegradation field is shown and discussed, aiming to analyze the combination of detoxification, devulcanization and polymer-cleavage mechanisms to achieve better degradation yields. The modified superficial structure of rubber materials after biological treatments might be an interesting way to reuse old rubber for re-vulcanization or to find new materials.\n",
      "\n",
      "title: Construction and Cloning of Plastic-degrading Recombinant Enzymes (MHETase).\n",
      "authors: ['Janatunaim RZ', 'Fibriani A']\n",
      "source: BACKGROUND: Polyethylene terephthalate (PET) is the most widely produced polyester plastic in the world. PET is very difficult to catalyze or biological depolymerization due to the limited access to ester bonds. Consequently, plastic will be stockpiled or flowed into the environment which is projected until hundreds of years. The most effective and environmental friendly plastic degradation method is biodegradation with microorganisms. Two specific enzyme for PET hydrolase, PETase and MHETase have been identified from Ideonella sakaiensis 201-F6. Recombinant genes are made to increase the effectiveness of enzymes in degrading PET. Previous studies of the PETase gene have been carried out, but to produce the final degradation PET product, the enzyme MHETase is needed. Thus, in this study the MHETase gene construction was carried out. METHODS: The goal of this study is to construct MHETase gene in pUCIDT plasmid with native signal peptide from I. sakaensis 201-F6 and constitutive promoter J23106 was expressed in Escherichia coli BL21 (DE3) by heats shock. Expression analysis using SDS-PAGE and activity of enzyme is analyzed by spectrophotometry method and SEM. RESULTS: MHETase gene protein was successfully constructed in pUCIDT +Amp plasmid with native signal peptide from Ideonella sakaensis 201-F6, T7 terminator and constitutive promoter J23106. PCR analysis showed that the gene successfully contained in the cells by band size (1813 bp) in electrophoresis gel. Analysis using Snap Gene, pairwise alignment using MEGA X, and NCBI was demonstrated that MHETase sequence the gene was in-frame in pUCIDT plasmid. CONCLUSION: MHETase gene was successfully constructed in plasmids by in silico method. Synthetic plasmids transformed in E. coli BL21 (DE3) contain MHETase gene sequences which were in frame. Hence, the E. coli BL21 (DE3) cells have the potential to produce MHETase proteins for the plastic degradation testing process. We will patent the construct of MHETase gene using constitutive promoter and signal peptide from native which expressed in E. coli BL21 (DE3). This patent refers to a more applicable plastic degradation system with a whole cell without the need for purification and environmental conditioning of pure enzymes.\n",
      "\n",
      "title: Perspectives and challenges of micro/nanoplastics-induced toxicity with special reference to phytotoxicity.\n",
      "authors: ['Maity S', 'Pramanick K']\n",
      "source: Plastic pollution has become a global concern for ecosystem health and biodiversity conservation. Concentrations of plastics are manifold higher in the terrestrial system than the aquatic one. Micro/nanoplastics (M/NP) have the ability to alter soil enzymatic system, soil properties and also affect soil borne microorganisms and earthworms. Despite, the knowhow regarding modulatory effects of plastics are acquired from the study on aquatic system and reports on their phytotoxic potentials are limited. The presence of cell wall that could restrict M/NP invasion into plant roots might be the putative cause of this limitation. M/NP inhibit plant growth, seed germination and gene expression; and they also induce cytogenotoxicity by aggravating reactive oxygen species generation. Dynamic behavior of cell wall; the pores formed either by cell wall degrading enzymes or by plant-pathogen interactions or by mechanical injury might facilitate the entry of into roots M/NP. This review also provides a possible mechanism of large sized microplastics-induced phytotoxicity especially for those that cannot pass through cell wall pores. As M/NP affect soil microbial community and soil parameters, it is hypothesized that they could have the potential to affect N2 fixation and research should be conducted in this direction. Reports on M/NP-induced toxicity mainly focused only on one polymer type (polystyrene) in spite of the toxicological relevancies of other polymer types like polyethylene, polypropylene etc. So, the assessment of phytotoxic potential of M/NP should be done using other plastic polymers in real environment as they are known to intract with other environmental stressors as well as can alter the the soil-microbe-plant interaction.\n",
      "\n",
      "title: A Novel Polyester Hydrolase From the Marine Bacterium Pseudomonas aestusnigri - Structural and Functional Insights.\n",
      "authors: ['Bollinger A', 'Thies S', 'Knieps-Grunhagen E', 'Gertzen C', 'Kobus S', 'Hoppner A', 'Ferrer M', 'Gohlke H', 'Smits SHJ', 'Jaeger KE']\n",
      "source: Biodegradation of synthetic polymers, in particular polyethylene terephthalate (PET), is of great importance, since environmental pollution with PET and other plastics has become a severe global problem. Here, we report on the polyester degrading ability of a novel carboxylic ester hydrolase identified in the genome of the marine hydrocarbonoclastic bacterium Pseudomonas aestusnigri VGXO14 (T) . The enzyme, designated PE-H, belongs to the type IIa family of PET hydrolytic enzymes as indicated by amino acid sequence homology. It was produced in Escherichia coli, purified and its crystal structure was solved at 1.09 A resolution representing the first structure of a type IIa PET hydrolytic enzyme. The structure shows a typical alpha/beta-hydrolase fold and high structural homology to known polyester hydrolases. PET hydrolysis was detected at 30 degrees C with amorphous PET film (PETa), but not with PET film from a commercial PET bottle (PETb). A rational mutagenesis study to improve the PET degrading potential of PE-H yielded variant PE-H (Y250S) which showed improved activity, ultimately also allowing the hydrolysis of PETb. The crystal structure of this variant solved at 1.35 A resolution allowed to rationalize the improvement of enzymatic activity. A PET oligomer binding model was proposed by molecular docking computations. Our results indicate a significant potential of the marine bacterium P. aestusnigri for PET degradation.\n",
      "\n",
      "title: Biology and applications of Clonostachys rosea.\n",
      "authors: ['Sun ZB', 'Li SD', 'Ren Q', 'Xu JL', 'Lu X', 'Sun MH']\n",
      "source: Clonostachys rosea is a promising saprophytic filamentous fungus that belongs to phylum Ascomycota. Clonostachys rosea is widespread around the world and exists in many kinds of habitats, with the highest frequency in soil. As an excellent mycoparasite, C. rosea exhibits strong biological control ability against numerous fungal plant pathogens, nematodes and insects. These behaviours are based on the activation of multiple mechanisms such as secreted cell-wall-degrading enzymes, production of antifungal secondary metabolites and induction of plant defence systems. Besides having significant biocontrol activity, C. rosea also functions in the biodegradation of plastic waste, biotransformation of bioactive compounds, as a bioenergy sources and in fermentation. This mini review summarizes information about the biology and various applications of C. rosea and expands on its possible uses.\n",
      "\n",
      "title: Production, purification and evaluation of biodegradation potential of PHB depolymerase of Stenotrophomonas sp. RZS7.\n",
      "authors: ['Sayyed RZ', 'Wani SJ', 'Alarfaj AA', 'Syed A', 'El-Enshasy HA']\n",
      "source: There are numerous reports on poly-beta-hydroxybutyrate (PHB) depolymerases produced by various microorganisms isolated from various habitats, however, reports on PHB depolymerase production by an isolate from plastic rich sites scares. Although PHB has attracted commercial significance, the inefficient production and recovery methods, inefficient purification of PHB depolymerase and lack of ample knowledge on PHB degradation by PHB depolymerase have hampered its large scale commercialization. Therefore, to ensure the biodegradability of biopolymers, it becomes imperative to study the purification of the biodegrading enzyme system. We report the production, purification, and characterization of extracellular PHB depolymerase from Stenotrophomonas sp. RZS7 isolated from a dumping yard rich in plastic waste. The isolate produced extracellular PHB depolymerase in the mineral salt medium (MSM) at 30 degrees C during 4 days of incubation under shaking. The enzyme was purified by three methods namely ammonium salt precipitation, column chromatography, and solvent purification. Among these purification methods, the enzyme was best purified by column chromatography on the Octyl-Sepharose CL-4B column giving optimum yield (0.7993 Umg-1mL-1). The molecular weight of purified PHB depolymerase was 40 kDa. Studies on the assessment of biodegradation of PHB in liquid culture medium and under natural soil conditions confirmed PHB biodegradation potential of Stenotrophomonas sp. RZS7. The results obtained in Fourier-Transform Infrared (FTIR) analysis, High-Performance Liquid Chromatography (HPLC) study and Gas Chromatography Mass-Spectrometry (GC-MS) analysis confirmed the biodegradation of PHB in liquid medium by Stenotrophomonas sp. RZS7. Changes in surface morphology of PHB film in soil burial as observed in Field Emission Scanning Electron Microscopy (FESEM) analysis confirmed the biodegradation of PHB under natural soil environment. The isolate was capable of degrading PHB and it resulted in 87.74% biodegradation. A higher rate of degradation under the natural soil condition is the result of the activity of soil microbes that complemented the biodegradation of PHB by Stenotrophomonas sp. RZS7.\n",
      "\n",
      "title: Fungal potential for the degradation of petroleum-based polymers: An overview of macro- and microplastics biodegradation.\n",
      "authors: ['Sanchez C']\n",
      "source: Petroleum-based plastic materials as pollutants raise concerns because of their impact on the global ecosystem and on animal and human health. There is an urgent need to remove plastic waste from the environment to overcome the environmental crisis of plastic pollution. This review describes the natural and unique ability of fungi to invade substrates by using enzymes that have the capacity to detoxify pollutants and are able to act on nonspecific substrates, the fungal ability to produce hydrophobins for surface coating to attach hyphae to hydrophobic substrates, and hyphal ability to penetrate three dimensional substrates. Fungal studies on macro- and microplastics biodegradation have shown that fungi are able to use these materials as the sole carbon and energy source. Further research is required on novel isolates from plastisphere ecosystems, on the use of molecular techniques to characterize plastic-degrading fungi and enhance enzymatic activity levels, and on the use of omics-based technologies to accelerate plastic waste biodegradation processes. The addition of pro-oxidants species (photosensitizers) and the reduction of biocides and antioxidant stabilizers used in the plastic manufacturing process should also be considered to promote biodegradation. Interdisciplinary research and innovative fungal strategies for plastic waste biodegradation, as well as ecofriendly manufacturing of petroleum-based plastics, may help to reduce the negative impacts of plastic waste pollution in the biosphere.\n",
      "\n",
      "title: Biodegradation of polyethylene microplastic particles by the fungus Aspergillus flavus from the guts of wax moth Galleria mellonella.\n",
      "authors: ['Zhang J', 'Gao D', 'Li Q', 'Zhao Y', 'Li L', 'Lin H', 'Bi Q', 'Zhao Y']\n",
      "source: Polyethylene (PE) products are widely used in daily life, agriculture, and industry because of their convenience and economic value. However, PE is one of the polymer materials remarkably resistant to degradation. Current methods of plastic waste disposal pose a threat to the environment and produce microplastic particles (MPP), which becomes a global environmental concern because of its accumulation. In this study, a PE-degrading fungus Aspergillus flavus named PEDX3, was isolated from the gut contents of wax moth Galleria mellonella. The results indicated that high-density polyethylene (HDPE) MPP was degraded into the MPP with a lower molecular weight by strain PEDX3 after 28 days incubation. In addition, Fourier Transform - Infrared Spectroscopy (FT-IR) results showed the appearance of carbonyl groups and ether groups of MPP, which also validated the degradation of PE. Furthermore, the potential degradation enzymes were investigated by Reverse Transcription-Polymerase Chain Reaction (RT-PCR). Finally, two laccase-like multicopper oxidases (LMCOs) genes, AFLA_006190 and AFLA_053930, displayed up-regulated expression during the degradation process, which may be the candidate PE-degrading enzymes. These results have demonstrated that the A. flavus strain PEDX3 has an ability to degrade microplastic particles and the two PE-degrading enzymes provide a promising application for the PE MPP remediation.\n",
      "\n",
      "title: RemeDB: Tool for Rapid Prediction of Enzymes Involved in Bioremediation from High-Throughput Metagenome Data Sets.\n",
      "authors: ['Sankara Subramanian SH', 'Balachandran KRS', 'Rangamaran VR', 'Gopal D']\n",
      "source: Environmental pollution has emerged to be a major hazard in today's world. Pollutants from varied sources cause harmful effects to the ecosystem. The major pollutants across marine and terrestrial regions are hydrocarbons, plastics, and dyes. Conventional methods for remediation have their own limitations and shortcomings to deal with these environmental pollutants. Bio-based remediation techniques using microbes have gained momentum in the recent past, primarily ascribed to their eco-friendly approach. The role of microbial enzymes in remediating the pollutants are well reported, and further exploration of microbial resources could lead to discovery of novel pollutant degrading enzymes (PDEs). Recent advances in next-generation sequencing technologies and metagenomics have provided the impetus to explore environmental microbes for potentially novel bioremediation enzymes. In this study, a tool, RemeDB, was developed for identifying bioremediation enzymes sequences from metagenomes. RemeDB aims at identifying hydrocarbon, dye, and plastic degrading enzymes from various metagenomic libraries. A sequence database consisting of >30,000 sequences proven to degrade the major pollutants was curated from various literature sources and this constituted the PDEs' database. Programs such as HMMER and RAPSearch were incorporated to scan across large metagenomic sequences libraries to identify PDEs. The tool was tested with metagenome data sets from varied sources and the outputs were validated. RemeDB was efficient to classify and identify the signature patterns of PDEs in the input data sets.\n",
      "\n",
      "title: Biodegradation and mineralization of polystyrene by plastic-eating superworms Zophobas atratus.\n",
      "authors: ['Yang Y', 'Wang J', 'Xia M']\n",
      "source: Polystyrene (PS) is one of the major plastic debris accumulated in environment. Previously, we reported that mealworm (Tenebrio molitor) was capable of degrading and mineralizing Styrofoam (PS foam). This finding arouses our curiosity to explore whether more other insect species have the same capability as mealworms. Here, an insect larva, superworm (Zophobas atratus), was newly proven to be capable of eating, degrading and mineralizing PS. Superworms could live with Styrofoam as sole diet as well as those fed with a normal diet (bran) over a 28-day period. The average consumption rate of Styrofoam for each superworm was estimated at 0.58 mg/d that was 4 times more than that of mealworm. Analyses of frass, using gel permeation chromatography (GPC), solid-state (13)C cross-polarization/magic angle spinning nuclear magnetic resonance (CP/MAS NMR) spectroscopy, and thermogravimetric interfaced with Fourier transform infrared (TG-FTIR) spectroscopy, demonstrated that the depolymerization of long-chain PS molecules and the formation of low molecular-weight products occurred in the larval gut. A respirometry test showed that up to 36.7% of the ingested Styrofoam carbon was converted into CO2 during a 16-day test period. The PS-degrading capability of superworm was inhibited by the antibiotic suppression of gut microbiota, indicating that gut microbiota contributed to PS degradation. This new finding extends the PS-degrading insects beyond the species within the Tenebrio genus and indicates that the gut microbiota of superworm would be a novel bioresource for pursuit of plastic-degrading enzymes.\n",
      "\n",
      "title: Biochemical properties and biotechnological applications of microbial enzymes involved in the degradation of polyester-type plastics.\n",
      "authors: ['Urbanek AK', 'Mironczuk AM', 'Garcia-Martin A', 'Saborido A', 'de la Mata I', 'Arroyo M']\n",
      "source: Application of polyester-degrading enzymes should be considered as an eco-friendly alternative to chemical recycling due to the huge plastic waste disposal nowadays. Many hydrolases from several fungi and bacteria have been discovered and successfully evaluated for their activity towards different aliphatic polyesters (PHA, PBS, PBSA, PCL, PLA), aromatic polyesters (PET, PBT, PMT) as well as their co-polyesters (PBST, PBAT, PBSTIL). This revision gives an up-to-date overview on the main biochemical features and biotechnological applications of those reported enzymes which are able to degrade polyester-based plastics, including different microbial polyester depolymerases, esterases, cutinase-like enzymes and lipases. Summarized information includes available protein sequences with the corresponding accession numbers deposited in NCBI server, 3D resolved structures, and data about optimal conditions for enzymatic activity and stability of many of these microbial enzymes that would be helpful for researchers in this topic. Although screening and identification of new native polyester hydrolases from microbial sources is undeniable according to literature, we briefly highlight the importance of the design of improved enzymes towards recalcitrant aromatic polyesters through different approaches that include site-directed mutagenesis and surface protein engineering.\n",
      "\n",
      "title: Changes in the gut microbiome and enzymatic profile of Tenebrio molitor larvae biodegrading cellulose, polyethylene and polystyrene waste.\n",
      "authors: ['Przemieniecki SW', 'Kosewska A', 'Ciesielski S', 'Kosewska O']\n",
      "source: Recent studies have demonstrated the ability of mealworm (Tenebrio molitor) for plastic degradation. This study is focused on changes in microbiome structure depending on diets. Microbial community obtained from oat and cellulose diet formed similar group, two kinds of polyethylene formed another group, while polystyrene diet showed the highest dissimilarity. The highest relative abundance of bacteria colonizing gut was in PE-oxodegradable feeding, nevertheless all applied diets were higher in comparison to oat. Dominant phyla consisted of Proteobacteria, Bacteroides, Firmicutes and Actinobacteria, however after PS feeding frequency in Planctomycetes and Nitrospirae increased. The unique bacteria characteristic for cellulose diet belonged to Selenomonas, while Pantoea were characteristic for both polyethylene diets, Lactococcus and Elizabethkingia were unique for each plastic diet, and potential diazotropic bacteria were characteristic for polystyrene diet (Agrobacterium, Nitrosomonas, Nitrospira). Enzymatic similarity between oatmeal and cellulose diets, was shown. All three plastics diet resulted in different activity in both, digestive tract and bacteria. The enzymes with the highest activity were included phosphatases, esterases, leucine arylamidase, beta-galactosidase, beta-glucuronidase, alpha-glucosidase, beta-glucosidase, chitinase, alpha-mannosidase and alpha-fucosidase. The activity of digestive tract was stronger than cultured gut bacteria. In addition to known polyethylene degradation methods, larvae may degrade polyethylene with esterase, cellulose and oatmeal waste activity is related with the activity of sugar-degrading enzymes, degradation of polystyrene with anaerobic processes and diazotrophs.\n",
      "\n",
      "title: In silico Screening and Heterologous Expression of a Polyethylene Terephthalate Hydrolase (PETase)-Like Enzyme (SM14est) With Polycaprolactone (PCL)-Degrading Activity, From the Marine Sponge-Derived Strain Streptomyces sp. SM14.\n",
      "authors: ['Almeida EL', 'Carrillo Rincon AF', 'Jackson SA', 'Dobson ADW']\n",
      "source: Plastics, such as the polyethylene terephthalate (PET), are widely used for various industrial applications, due to their physicochemical properties which are particularly useful in the packaging industry. However, due to improper plastic waste management and difficulties in recycling, post-consumer plastic waste has become a pressing issue for both the environment and for human health. Hence, novel technologies and methods of processing plastic waste are required to address these issues. Enzymatic-assisted hydrolysis of synthetic polymers has been proposed as a potentially more efficient and environment-friendly alternative to the currently employed methods. Recently, a number of PET hydrolases have been described, and in particular a PETase derived from Ideonella sakaiensis 201-F6 (IsPETase), which appears to be the most efficient and substrate-specific bacterial PET hydrolase enzyme discovered to date. In order to further investigate this class of PETase-like enzymes, we employed an in silico-based screening approach on the biotechnologically relevant genus Streptomyces, including terrestrial and marine isolates; in a search for potential PETase homologs. From a total of 52 genomes analyzed, we were able to identify three potential PETase-like enzymes, all of which were derived from marine-sponge associated Streptomyces isolates. A candidate PETase-like gene (SM14est) was identified in Streptomyces sp. SM14. Further in silico characterization of the SM14est protein sequence and its predicted three-dimensional structure were performed and compared to the well-characterized IsPETase. Both the serine hydrolase motif Gly-x1-Ser-x2-Gly and the catalytic triad Ser, Asp, His are conserved in both sequences. Molecular docking experiments indicated that the SM14est enzyme possessed the capacity to bind plastics as substrates. Finally, polyesterase activity was confirmed using a polycaprolactone (PCL) plate clearing assay which is a model substrate for the degradation of plastics; following heterologous expression of SM14est in Escherichia coli, with secretion being facilitated by the native Streptomyces signal peptide. These findings provide further insights into this important class of PETase-like enzymes.\n",
      "\n",
      "title: Enzymatic PET Degradation.\n",
      "authors: ['Papadopoulou A', 'Hecht K', 'Buller R']\n",
      "source: Plastic, in the form of packaging material, disposables, clothing and other articles with a short lifespan, has become an indispensable part of our everyday life. The increased production and use of plastic, however, accelerates the accumulation of plastic waste and poses an increasing burden on the environment with negative effects on biodiversity and human health. PET, a common thermoplastic, is recycled in many countries via thermal, mechanical and chemical means. Recently, several enzymes have been identified capable of degrading this recalcitrant plastic, opening possibilities for the biological recycling of the omnipresent material. In this review, we analyze the current knowledge of enzymatic PET degradation and discuss advances in improving the involved enzymes via protein engineering. Looking forward, the use of plastic degrading enzymes may facilitate sustainable plastic waste management and become an important tool for the realization of a circular plastic economy.\n",
      "\n",
      "title: Disentangling loosening from softening: insights into primary cell wall structure.\n",
      "authors: ['Zhang T', 'Tang H', 'Vavylonis D', 'Cosgrove DJ']\n",
      "source: How cell wall elasticity, plasticity, and time-dependent extension (creep) relate to one another, to plant cell wall structure and to cell growth remain unsettled topics. To examine these issues without the complexities of living tissues, we treated cell-free strips of onion epidermal walls with various enzymes and other agents to assess which polysaccharides bear mechanical forces in-plane and out-of-plane of the cell wall. This information is critical for integrating concepts of wall structure, wall material properties, tissue mechanics and mechanisms of cell growth. With atomic force microscopy we also monitored real-time changes in the wall surface during treatments. Driselase, a potent cocktail of wall-degrading enzymes, removed cellulose microfibrils in superficial lamellae sequentially, layer-by-layer, and softened the wall (reduced its mechanical stiffness), yet did not induce wall loosening (creep). In contrast Cel12A, a bifunctional xyloglucanase/cellulase, induced creep with only subtle changes in wall appearance. Both Driselase and Cel12A increased the tensile compliance, but differently for elastic and plastic components. Homogalacturonan solubilization by pectate lyase and calcium chelation greatly increased the indentation compliance without changing tensile compliances. Acidic buffer induced rapid cell wall creep via endogenous alpha-expansins, with negligible effects on wall compliances. We conclude that these various wall properties are not tightly coupled and therefore reflect distinctive aspects of wall structure. Cross-lamellate networks of cellulose microfibrils influenced creep and tensile stiffness whereas homogalacturonan influenced indentation mechanics. This information is crucial for constructing realistic molecular models that define how wall mechanics and growth depend on primary cell wall structure.\n",
      "\n",
      "title: Development of a safe and efficient gene delivery system based on a biodegradable tannic acid backbone.\n",
      "authors: ['Wu S', 'Yan Y', 'Ni D', 'Pan X', 'Chen X', 'Guan J', 'Xiong X', 'Liu L']\n",
      "source: Finding a safe and efficient gene delivery vector is a major international challenge facing the development of gene therapy. Tannic acid (TA) is a natural cross-linker owing to its hydroxyl and carboxyl groups that can interact with biopolymers for different biomaterial design. In this work, three polyethyleneimine-modified TA polymers were prepared, and the polymers were characterized by FTIR, UV-vis, elemental analysis and (1)H NMR. The potential of PTAs as gene vector was studied in vitro, including DNA loading capacity, DNA protection ability and biocompatibility. In addition, the particle size, zeta potential, DNA encapsulation efficiency, cell uptake and transfection efficiency of the PTA-pDNA polyplexes were also studied. The results showed that PTA2k and PTA30k could completely condense DNA at N/P of 2, and PTA600 could only completely condense DNA at N/P of 50. The PTA/pDNA polyplexes could protect DNA from degrading by DNA enzymes and could be efficiently uptaked by cells. Biocompatibility assay showed that PTA had no significant cytotoxicity and effect on cell proliferation compared to PEI. At low N/P ratios of 1-4, PTA showed higher transfection efficiency than PEI, and the transfection efficiency increased with the increase of PEI molecular weight in PTA. At N/P of 3, PTA30k showed the highest transfection efficiency of 23.8%, while PEI30k showed only 6.7%. These results indicate that PTA is a promising candidate vector for safe and efficient gene delivery.\n",
      "\n",
      "title: Plastics: Environmental and Biotechnological Perspectives on Microbial Degradation.\n",
      "authors: ['Danso D', 'Chow J', 'Streit WR']\n",
      "source: Plastics are widely used in the global economy, and each year, at least 350 to 400 million tons are being produced. Due to poor recycling and low circular use, millions of tons accumulate annually in terrestrial or marine environments. Today it has become clear that plastic causes adverse effects in all ecosystems and that microplastics are of particular concern to our health. Therefore, recent microbial research has addressed the question of if and to what extent microorganisms can degrade plastics in the environment. This review summarizes current knowledge on microbial plastic degradation. Enzymes available act mainly on the high-molecular-weight polymers of polyethylene terephthalate (PET) and ester-based polyurethane (PUR). Unfortunately, the best PUR- and PET-active enzymes and microorganisms known still have moderate turnover rates. While many reports describing microbial communities degrading chemical additives have been published, no enzymes acting on the high-molecular-weight polymers polystyrene, polyamide, polyvinylchloride, polypropylene, ether-based polyurethane, and polyethylene are known. Together, these polymers comprise more than 80% of annual plastic production. Thus, further research is needed to significantly increase the diversity of enzymes and microorganisms acting on these polymers. This can be achieved by tapping into the global metagenomes of noncultivated microorganisms and dark matter proteins. Only then can novel biocatalysts and organisms be delivered that allow rapid degradation, recycling, or value-added use of the vast majority of most human-made polymers.\n",
      "\n",
      "title: Biodegradation of polyvinyl alcohol by different dominant degrading bacterial strains in a baffled anaerobic bioreactor.\n",
      "authors: ['Wu HF', 'Yue LZ', 'Jiang SL', 'Lu YQ', 'Wu YX', 'Wan ZY']\n",
      "source: Polyvinyl alcohol (PVA) is the main pollutant in printing and dyeing wastewaters. This pollutant exhibits great demand, poor biodegradability and refractory degradation. In this study, PVA wastewater treatment experiments were conducted in a stably operating baffled anaerobic bioreactor (ABR) by using simulated PVA wastewater. The PVA degradation pathway and mechanism of the mixed dominant PVA-degrading bacterial strains were identified through the analysis of their degradation products. From the results, we inferred that PVA was degraded in a stepwise process under the synergistic action of different extracellular and intracellular enzymes produced by the mixed dominant PVA-degrading bacterial strains. In this process, PVA was first degraded into ketones, fatty acids and alcohols. It was then regenerated into acetic acid, hydrogen and carbon dioxide. Finally, these substances could be further utilized by methanogens. PVA was thus degraded completely. This study may serve as a reference for future works on the degradation of PVA in the ecological environment. It may also guide the sustainable development of PVA.\n",
      "\n",
      "title: Deficiency of biodegradable plastic-degrading enzyme production in a gene-deletion mutant of phyllosphere yeast, Pseudozyma antarctica defective in mannosylerythritol lipid biosynthesis.\n",
      "authors: ['Saika A', 'Koike H', 'Yarimizu T', 'Watanabe T', 'Kitamoto H', 'Morita T']\n",
      "source: The basidiomycetous yeast Pseudozyma antarctica (currently designated Moesziomyces antarcticus) produces extracellular enzymes and glycolipids, including mannosylerythritol lipids (MELs), which are biosurfactants. Strain GB-4(0) of this species was previously isolated from rice husks and produces biodegradable plastic-degrading enzyme (Pseudozyma antarctica esterase; PaE). In this study, we generated a MEL biosynthesis-deficient strain (PaEMT1) by deleting the gene PaEMT1, which is essential to MEL biosynthesis in strain GB-4(0). The resulting PaEMT1 strain showed deficient PaE activity, and the corresponding signal was hardly detected in its culture supernatant through western blotting analysis using rabbit anti-PaE serum. On the other hand, the relative expression of the gene PaCLE1, encoding PaE, was identical between GB-4(0) and PaEMT1 based on quantitative real-time PCR. When strain PaEMT1 was grown in culture media supplemented with various surfactants, i.e., Tween20, BRIJ35 and TritonX-100, and MELs, PaE activity and secretion recovered. We also attempted to detect intracellular PaE using cell-free extract, but observed no signal in the soluble or insoluble fractions of PaEMT1. This result suggested that the PaCLE1 gene was not translated to PaE, or that expressed PaE was degraded immediately in PaEMT1. Based on these results, MEL biosynthesis is an important contributor to PaE production.\n",
      "\n",
      "title: Biodegradation of Structurally Diverse Phthalate Esters by a Newly Identified Esterase with Catalytic Activity toward Di(2-ethylhexyl) Phthalate.\n",
      "authors: ['Huang H', 'Zhang XY', 'Chen TL', 'Zhao YL', 'Xu DS', 'Bai YP']\n",
      "source: Herein, we report a double enzyme system to degrade 12 phthalate esters (PAEs), particularly bulky PAEs, such as the widely used bis(2-ethylhexyl) phthalate (DEHP), in a one-pot cascade process. A PAE-degrading bacterium, Gordonia sp. strain 5F, was isolated from soil polluted with plastic waste. From this strain, a novel esterase (GoEst15) and a mono(2-ethylhexyl) phthalate hydrolase (GoEstM1) were identified by homology-based cloning. GoEst15 showed broad substrate specificity, hydrolyzing DEHP and 10 other PAEs to monoalkyl phthalates, which were further degraded by GoEstM1 to phthalic acid. GoEst15 and GoEstM1 were heterologously coexpressed in Escherichia coli BL21 (DE3), which could then completely degrade 12 PAEs (5 mM), within 1 and 24 h for small and bulky substrates, respectively. To our knowledge, GoEst15 is the first DEHP hydrolase with a known protein sequence, which will enable protein engineering to enhance its catalytic performance in the future.\n",
      "\n",
      "title: Promotion of the growth and plant biomass degrading enzymes production in solid-state cultures of Lentinula edodes expressing Vitreoscilla hemoglobin gene.\n",
      "authors: ['Wang X', 'Ding Y', 'Gao X', 'Liu H', 'Zhao K', 'Gao Y', 'Qiu L']\n",
      "source: Vitreoscilla hemoglobin (VHb), encoded by the Vitreoscilla hemoglobin gene (vgb), is highly effective at binding oxygen and delivering it to both prokaryotes and eukaryotes under hypoxic conditions. In this study, we introduced the vgb gene into shiitake mushrooms, and the mycelia of the transformatants grew faster. In particular, they spread into the solid substrate located in the lower part of the test tubes and bags where the oxygen was hypoxic and produced more beta-glucan and plant biomass degrading enzymes compared to the original strain. The maximum growth rate of the transformants was 8.5%-15.9% higher than that of the original strain on sawdust-based cultures in plastic bags. The laccase and amylase activities were 17.7%-40.3% and 16.7%-37.9% higher than that of the original strain, respectively. In addition, the beta-glucan contents of the transformant mycelia from the submerged fermentation were 12.9%-24.0% higher than that of the original strain. These results reveal that the expression of VHb in mushroom fungi promots the mycelial growth in solid-state cultures under the hypoxic condition as well as enhances beta-glucan and plant biomass degrading enzymes production.\n",
      "\n",
      "title: Purification and kinetics of the PHB depolymerase of Microbacterium paraoxydans RZS6 isolated from a dumping yard.\n",
      "authors: ['Sayyed RZ', 'Wani SJ', 'Alyousef AA', 'Alqasim A', 'Syed A', 'El-Enshasy HA']\n",
      "source: Poly-beta-hydroxybutyrate (PHB) depolymerase is known to decompose PHB, biodegradable polymers and therefore has great commercial significance in the bioplastic sector. However, reports on PHB depolymerases from isolates obtained from plastic-contaminated sites that reflect the potential of the source organism is scarce. In this study, we evaluated the production of extracellular PHB depolymerase from Microbacterium paraoxydans RZS6 isolated from the plastic-contaminated site in the municipal area of Shahada, Maharashtra, India, for the first time. The isolate was identified using 16S rRNA gene sequencing, gas chromatographic analysis of fatty acid methyl esters (GC-FAME), and BIOLOG method. Ithydrolyzed PHB on minimal salt medium (MSM) containing PHB as the only source of carbon. The isolate produced PHB depolymerase at 45 degrees C during 48 h of incubation. The enzyme was purified most efficiently using octyl-sepharose CL-4B column, with the highest purification yield of 6.675 Umg-1mL-1. The activity of the enzyme was enhanced in the presence of Ca2+ and Mg2+ ions but inhibited by Fe2+ (1 mM) ions and mercaptoethanol (1000 rpm). the nzyme kinetic analysis revealed that the enzyme was a metalloenzyme; requiring Mg2+ ions, that showed optimum enzyme activity at 30 degrees C (mesophilic) and under neutrophilic (pH 7) conditions. Scale-up from the shake-flask level to a laboratory-scale bioreactor further enhanced the enzyme yield by 0.809 UmL-1. The molecular weight of the enzyme (40 kDa), as estimated by sodium dodecyl sulfate-polyacrylamide gel electrophoresis, closely resembled the PHB depolymerase of Aureobacterium saperdae. Our findings highlighted the applicability of M. paraoxydans as a producer of extracellular PHB depolymerase having potential of degrading PHB under diverse conditions.\n",
      "\n",
      "title: CRISPR/Cas9-mediated gene replacement in the basidiomycetous yeast Pseudozyma antarctica.\n",
      "authors: ['Kunitake E', 'Tanaka T', 'Ueda H', 'Endo A', 'Yarimizu T', 'Katoh E', 'Kitamoto H']\n",
      "source: The basidiomycetous yeast, Pseudozyma antarctica, has the ability to express industrially beneficial biodegradable plastic-degrading enzyme (PaE) and glycolipids. In this study, we developed a highly efficient gene-targeting method in P. antarctica using a CRISPR/Cas9 gene-editing approach. Transformation of protoplast cells was achieved by incubation with a ribonucleoprotein (RNP) complex prepared by mixing the Cas9 protein with a single-guide RNA together with donor DNA (dDNA) containing a selectable marker in vitro. The PaE gene was selected as the targeted locus for gene disruption and gene-disrupted colonies were readily detected by their ability to degrade polybutylene succinate-co-adipate on solid media. The accuracy of the gene conversion event was confirmed by colony PCR. An increase in the RNP mix increased both transformation and gene disruption efficiencies. Examining the effect of the homology arm length of the dDNA revealed that dDNA with homology arms longer than 0.1kb induced efficient homologous recombination in our system. Furthermore, this system was successful in another targeted locus, PaADE2. Following the creation of RNP-induced double-strand break of the chromosomal DNA, dDNA could be inserted into the target locus even in the absence of homology arms.\n",
      "\n",
      "title: Ultrahigh-throughput screening system for directed polymer binding peptide evolution.\n",
      "authors: ['Apitius L', 'Rubsam K', 'Jakesch C', 'Jakob F', 'Schwaneberg U']\n",
      "source: Accumulation of plastics in the environment became a geological indicator of the Anthropocene era. An effective reduction of long-lasting plastics requires a treatment with micro-organisms that release polymer-degrading enzymes. Polymer binding peptides function as adhesion promoters and enable a targeted binding of whole cells to polymer surfaces. An esterase A-based Escherichia coli cell surface display screening system was developed, that enabled directed evolution of polymer binding peptides for improved binding strength to polymers. The E. coli cell surface screening system facilitates an enrichment of improved binding peptides from a culture broth through immobilization of whole cells on polymer beads. The polypropylene (PP)-binding peptide liquid chromatography peak I (LCI) was simultaneously saturated at five positions (Y29, D31, G35, E42, and D45; 3.2 million variants) and screened for improved PP-binding in the presence of the anionic surfactant sodium dodecylbenzenesulfonate (LAS; 0.25 mM). The cell surface system enabled efficient screening of the generated LCI diversity (in total ~10 million clones were screened). Characterization of identified LCI binders revealed an up to 12-fold improvement (eGFP-LCI-CSD-3: E42V/D45H) in PP-binding strength in the presence of the surfactant LAS (0.125 mM). The latter represents a first whole cell display screening system to improve adhesion peptides which can be used to direct and to immobilize organisms specifically to polymer surfaces (e.g., PP) and novel applications (e.g., in targeted plastic degradation).\n",
      "\n",
      "title: Structure of the plastic-degrading Ideonella sakaiensis MHETase bound to a substrate.\n",
      "authors: ['Palm GJ', 'Reisky L', 'Bottcher D', 'Muller H', 'Michels EAP', 'Walczak MC', 'Berndt L', 'Weiss MS', 'Bornscheuer UT', 'Weber G']\n",
      "source: The extreme durability of polyethylene terephthalate (PET) debris has rendered it a long-term environmental burden. At the same time, current recycling efforts still lack sustainability. Two recently discovered bacterial enzymes that specifically degrade PET represent a promising solution. First, Ideonella sakaiensis PETase, a structurally well-characterized consensus alpha/beta-hydrolase fold enzyme, converts PET to mono-(2-hydroxyethyl) terephthalate (MHET). MHETase, the second key enzyme, hydrolyzes MHET to the PET educts terephthalate and ethylene glycol. Here, we report the crystal structures of active ligand-free MHETase and MHETase bound to a nonhydrolyzable MHET analog. MHETase, which is reminiscent of feruloyl esterases, possesses a classic alpha/beta-hydrolase domain and a lid domain conferring substrate specificity. In the light of structure-based mapping of the active site, activity assays, mutagenesis studies and a first structure-guided alteration of substrate specificity towards bis-(2-hydroxyethyl) terephthalate (BHET) reported here, we anticipate MHETase to be a valuable resource to further advance enzymatic plastic degradation.\n",
      "\n",
      "title: Downregulation of RNF128 activates Wnt/beta-catenin signaling to induce cellular EMT and stemness via CD44 and CTTN ubiquitination in melanoma.\n",
      "authors: ['Wei CY', 'Zhu MX', 'Yang YW', 'Zhang PF', 'Yang X', 'Peng R', 'Gao C', 'Lu JC', 'Wang L', 'Deng XY', 'Lu NH', 'Qi FZ', 'Gu JY']\n",
      "source: BACKGROUND: Ring finger proteins (RNFs) were involved in carcinogenesis. Here, we aimed to explore the detailed mechanism of RNF128 in the progression of melanoma. METHODS: We reanalyzed several gene expression profiles from the Gene Expression Omnibus (GEO) database and obtained the overlapped differential expressed RNF genes. Among them, RNF128 was selected to further explore its expression, the biological significance, and the underlying molecular mechanism, as well as the clinical relevance in melanoma patients. RESULTS: RNF128 was found to be significantly downregulated in the selected datasets, which was further verified in our melanoma tissues. Moreover, RNF128 downregulation was shown to correlate with the malignant phenotype of melanoma, and further functional assays demonstrated that low levels of RNF128 promoted melanoma progression via inducing cell epithelial-mesenchymal transition (EMT) and the acquisition of stemness. Mechanistically, RNF128 interference activated the Wnt pathway via simultaneously ubiquitinating CD44/cortactin (CTTN), resulting in CD44 and c-Myc transcription, thus revealed that RNF128 participated in a positive feedback of the Wnt pathway-CD44 loop. Clinically, we found that patients expressing low RNF128 and high CD44/CTTN levels had a poor prognosis. CONCLUSION: Downregulated RNF128 activates Wnt signaling to induce cellular EMT and stemness by ubiquitinating and degrading CD44/CTTN, and RNF128 is a reliable diagnostic and prognostic biomarker, and a deeper understanding of RNF128 may contribute to the treatment of melanoma.\n",
      "\n",
      "title: Isolation of a polyethylene degrading Paenibacillus sp. from a landfill in Brazil.\n",
      "authors: ['Bardaji DKR', 'Furlan JPR', 'Stehling EG']\n",
      "source: The annual production of plastics has doubled over the past 15 years and, consequently, a large amount of plastic has accumulated in the environment generating ecological problems. In this study, a Paenibacillus sp. isolate was obtained from a landfill from Brazil and it presented the alkane hydroxylase gene (alkB). Weight loss of low-density polyethylene (LDPE) was measured and a significant difference in final weight compared to initial weight was assessed. Some chemical characteristics, such as bond scissions and formation of new functional groups [carboxylic acids (3300-2500 cm(-1)), esters (1210-1163 cm(-1)), and ethers (1075-1020 cm(-1))], were detected by Fourier-transform infrared spectroscopy. Bacterial colonization on the plastic surface and physical changes, as formation of cracks and pits, was visualized by scanning electron microscopy. This isolate was susceptible to all the antimicrobials tested. Therefore, this isolate possesses great potential to degrade polyethylene and become an option for LDPE bioremediation.\n",
      "\n",
      "title: Enzymatic recycling of thermoplastic polyurethanes: Synergistic effect of an esterase and an amidase and recovery of building blocks.\n",
      "authors: ['Magnin A', 'Pollet E', 'Perrin R', 'Ullmann C', 'Persillon C', 'Phalip V', 'Averous L']\n",
      "source: Biological recycling of polyurethanes (PU) is a huge challenge to take up in order to reduce a large part of the environmental pollution from these materials. However, enzymatic depolymerization of PU still needs to be improved to propose valuable and green solutions. The present study aims to identify efficient PU degrading enzymes among a collection of 50 hydrolases. Screenings based on model molecules were performed leading to the selection of an efficient amidase (E4143) able to hydrolyze the urethane bond of a low molar mass molecule and an esterase (E3576) able to hydrolyze a waterborne polyester polyurethane dispersion. Degradation activities of the amidase, the esterase and a mix of these enzymes were then evaluated on four thermoplastic polyurethanes (TPU) specifically designed for this assay. The highest degradation was obtained on a polycaprolactone polyol-based polyurethane with weight loss of 33% after 51days measured for the esterase. Deep cracks on the polymer surface observed by scanning electron microscopy and the presence of oligomers on the remaining TPU detected by size exclusion chromatography evidenced the polymer degradation. Mixing both enzymes led to an increased amount of urethane bonds hydrolysis of the polymer. 6-hydroxycaproic acid and 4,4'-methylene dianiline were recovered after depolymerization as hydrolysis products. Such building blocks could get a second life with the synthesis of new macromolecular architectures.\n",
      "\n",
      "title: Structural insights into the unique polylactate-degrading mechanism of Thermobifida alba cutinase.\n",
      "authors: ['Kitadokoro K', 'Kakara M', 'Matsui S', 'Osokoshi R', 'Thumarat U', 'Kawai F', 'Kamitani S']\n",
      "source: Cutinases are enzymes known to degrade polyester-type plastics. Est119, a plastic-degrading type of cutinase from Thermobifida alba AHK119 (herein called Ta_cut), shows a broad substrate specificity toward polyesters, and can degrade substrates including polylactic acid (PLA). However, the PLA-degrading mechanism of cutinases is still poorly understood. Here, we report the structure complexes of cutinase with ethyl lactate (EL), the constitutional unit. From this complex structure, the electron density maps clearly showed one lactate (LAC) and one EL occupying different positions in the active site cleft. The binding mode of EL is assumed to show a figure prior to reaction and LAC is an after-reaction product. These complex structures demonstrate the role of active site residues in the esterase reaction and substrate recognition. The complex structures were compared with other documented complex structures of cutinases and with the structure of PETase from Ideonella sakaiensis. The amino acid residues involved in substrate interaction are highly conserved among these enzymes. Thus, mapping the precise interactions in the Ta_cut and EL complex will pave the way for understanding the plastic-degrading mechanism of cutinases and suggest ways of creating more potent enzymes by structural protein engineering.\n",
      "\n",
      "title: Construction of a Pseudozyma antarctica strain without foreign DNA sequences (self-cloning strain) for high yield production of a biodegradable plastic-degrading enzyme.\n",
      "authors: ['Sameshima-Yamashita Y', 'Watanabe T', 'Tanaka T', 'Tsuboi S', 'Yarimizu T', 'Morita T', 'Koike H', 'Suzuki K', 'Kitamoto H']\n",
      "source: The basidiomycetous yeast Pseudozyma antarctica GB-4(0) esterase (PaE) is a promising candidate for accelerating degradation of used biodegradable plastics (BPs). To increase safety and reduce costs associated with the use of PaE, we constructed a self-cloning strain with high-PaE productivity. A Lys12 gene (PaLYS12)-deleted lysine auxotroph strain GB4-(0)-L1 was obtained from GB-4(0) by ultraviolet mutagenesis and nystatin enrichment. Subsequently, the PaE gene (PaCLE1) expression cassette consisting of GB-4(0)-derived PaCLE1, under the control of a xylose-inducible xylanase promoter with PaLYS12, was randomly introduced into the GB4-(0)-L1 genome. A PaE high-producing strain, PGB474, was selected from among the transformants by high throughput double-screening based on its ability to degrade emulsified polybutylene succinate-co-adipate. Quantitative PCR revealed that four copies of the PaE gene expression cassette were introduced into the PGB474 genome. PGB474 produced 2.0 g/L of PaE by xylose-fed-batch cultivation using a 3-L jar fermentor for 72 h.\n",
      "\n",
      "title: Immobilization of antimicrobial and anti-quorum sensing enzymes onto GMA-grafted poly(vinyl chloride) catheters.\n",
      "authors: ['Costoya A', 'Velazquez Becerra LE', 'Melendez-Ortiz HI', 'Diaz-Gomez L', 'Mayer C', 'Otero A', 'Concheiro A', 'Bucio E', 'Alvarez-Lorenzo C']\n",
      "source: Catheter-associated infections still represent a challenging thread because of the likelihood of biofilm formation. The aim of this work was the surface modification of catheters to immobilize lysozyme and acylase under mild conditions while preserving antimicrobial and anti-quorum sensing performances. Glycidyl methacrylate (GMA) was grafted onto poly(vinyl chloride) (PVC) catheters by a pre-irradiation method. The effects of monomer concentration, pre-irradiation dose, reaction time, monomer concentration and reaction temperature were investigated. The grafting process was monitored using FTIR-ATR spectroscopy, differential scanning calorimetry, thermogravimetric analysis, and swelling data. Lysozyme was directly immobilized onto PVC-g-GMA maintaining the hydrolytic activity, which hindered Staphylococcus aureus adhesion. For acylase immobilization, the PVC-g-GMA catheters were reacted with ethylenediamine and glutaraldehyde in order to facilitate acylase covalent binding. Free acylase in solution demonstrated notably capability to act as quorum sensing inhibitor, as observed using Chromobacterium violaceum as biosensor, by degrading a wide variety of acylated homoserine lactones (AHLs), including those produced by Pseudomonas aeruginosa and Acinetobacter baumannii. Acylase-immobilized PVC-g-GMA catheters were challenged against degradation of AHLs and the activity monitored using both the biosensor and HPLC-MS. Relevantly, the functionalized catheters completely degraded all tested AHL signals, opening new ways of preventing biofilm formation on medical devices.\n",
      "\n",
      "title: Isolation and characterization of different promising fungi for biological waste management of polyurethanes.\n",
      "authors: ['Magnin A', 'Hoornaert L', 'Pollet E', 'Laurichesse S', 'Phalip V', 'Averous L']\n",
      "source: As a highly resistant polymer family, polyurethanes (PU) are responsible for increasing environmental issues. Then, PU biodegradation is a challenging way to develop sustainable waste management processes based on biological recycling. Since the metabolic diversity of fungi is a major asset for polymer degradation, nearly thirty strains were isolated from sampling on six different PU wastes-containing environments. A screening of the fungi on four thermoplastic PU (TPU) with different macromolecular architectures led to the selection of three strains able to use two polyester PU as sole carbon source: Alternaria sp., Penicillium section Lanata-Divaricata and Aspergillus section flavi. Weight loss, FT-IR, Scanning Electron Microscopy and Size Exclusion Chromatography analyses revealed that these three fungi degrade slightly and similarly a fatty acid dimer-based TPU while variability of degradation was noticed on a polycaprolactone-based TPU. On this last TPU, robust analysis of the degraded polymers showed that the Penicillium strain was the best degrading microorganism. Membrane enzymes seemed to be involved in this degradation. It is the first time that a strain of Penicillium of the section Lanata-Divaricata displaying PU biodegradation ability is isolated. These newly discovered fungi are promising for the development of polyester PU waste management process.\n",
      "\n",
      "title: Biodegradation of polyvinyl alcohol using cross-linked enzyme aggregates of degrading enzymes from Bacillus niacini.\n",
      "authors: ['Bian H', 'Cao M', 'Wen H', 'Tan Z', 'Jia S', 'Cui J']\n",
      "source: In this study, polyvinyl alcohol (PVA)-degrading bacteria were screened from sludge samples using PVA as a sole source of carbon. A novel strain was obtained and identified as Bacillus niacini based on the analysis of a partial 16S rDNA nucleotide sequence and morphological characteristics. PVA-degrading enzyme (PVAase) from Bacillus niacini was immobilized as cross-linked enzyme aggregates (CLEAs) via precipitation with ammonium sulfate followed by glutaraldehyde cross-linking. The effects of precipitation and cross-linking on PVAase-CLEAs activity were investigated and characterized. 70% ammonium sulfate and 1.5% glutaraldehyde were used for precipitation and 1-h cross-linking reaction. The activity recovery of PVAase-CLEAs was approximately 90% starting from free PVAase, suggesting non-purification steps are required for extended use. No significant differences in optimum pH and temperature values of the PVAase were recorded after immobilization. The PVAase-CLEAs showed a ball-like morphology and enhanced PVA degradation efficiency in comparison with the free PVAase in solution. Furthermore, the PVAase-CLEAs exhibited excellent thermal stability, pH stability and storage stability compared to free PVAase. The PVAase-CLEAs retained about 75% of initial PVAase activity after 4cycles of use. These results suggest that this CLEA is potentially usable for PVA degradation in industrial applications.\n",
      "\n",
      "title: The Collagenase of the Bacterium Clostridium histolyticum in the Treatment of Irradiation-Induced Capsular Contracture.\n",
      "authors: ['Diehm YF', 'Hirche C', 'Berger MR', 'Heil J', 'Golatta M', 'Kotsougiani D', 'Pomahac B', 'Kneser U', 'Fischer S']\n",
      "source: BACKGROUND: Irradiation therapy is an important pillar in the treatment of breast cancer. However, it can trigger capsular fibrosis, the most significant complication of implant-based breast reconstruction. As collagen is the main component of fibrotic capsules, the collagenase of the bacterium Clostridium histolyticum poses a potential treatment option for this pathological condition. METHODS: Thirty-six rats received miniature silicone implants on their backs. On day 1, the implant sites of two groups were irradiated with 10 Gy. On day 120, one irradiated group received collagenase injections into the implant pockets (n = 12). Non-irradiated (n = 12) and irradiated capsules (n = 12) were injected with plain solvent solution serving as controls. Data were analyzed by means of in vivo imaging, histology, immunohistochemistry and gene expression analysis. RESULTS: Compared with both controls, the injection of collagenase led to significantly thinner capsules. This was verified by in vivo imaging and histology. Although irradiation provoked alterations in capsule collagen structure and vessel wall thickness, the application of collagenase resulted in a significant reduction of collagen density. This was accompanied by an up-regulation of VEGF-A gene expression. Of note, hematoma formation inside the implant pocket occurred in two cases after collagenase injection. CONCLUSIONS: The collagenase of the bacterium Clostridium histolyticum is effective in degrading irradiation-induced capsular fibrosis around silicone implants. Hematoma formation occurred most likely because of irradiation-induced alterations in vessel wall architecture and capsule vascularization. Further studies need to be performed to address the clinical safety of this novel treatment option.\n",
      "\n",
      "title: Fast Turbidimetric Assay for Analyzing the Enzymatic Hydrolysis of Polyethylene Terephthalate Model Substrates.\n",
      "authors: ['Belisario-Ferrari MR', 'Wei R', 'Schneider T', 'Honak A', 'Zimmermann W']\n",
      "source: Synthetic plastics such as polyethylene terephthalate (PET) can be cooperatively degraded by microbial polyester hydrolases and carboxylesterases, with the latter hydrolyzing the low-molecular-weight degradation intermediates. For the identification of PET-degrading enzymes, efficient and rapid screening assays are required. Here a novel turbidimetric method in a microplate format for the fast screening of enzyme activities against the PET model substrates with two ester bonds bis-(2-hydroxyethyl) terephthalate (BHET) and ethylene glycol bis-(p-methylbenzoate) (2PET) is reported. The carboxylesterase TfCa from Thermobifida fusca KW3 is used for validating the method. High correlation and regression coefficients between the experimental and fitted data confirm the accuracy and reproducibility of the method and its feasibility for analyzing the kinetics of the enzymatic hydrolysis of the PET model substrates. A comparison of the hydrolysis of BHET and 2PET by TfCa using a kinetic model for heterogeneous catalysis indicates that the enzyme preferentially hydrolyzes the less bulky molecule BHET. The high-throughput assay will facilitate the detection of novel enzymes for the biocatalytic modification or degradation of PET.\n",
      "\n",
      "title: Purification and characterization of new bio-plastic degrading enzyme from Burkholderia cepacia DP1.\n",
      "authors: ['Azura Azami N', 'Ira Aryani W', 'Aik-Hong T', 'Amirul AA']\n",
      "source: Depolymerase is an enzyme that plays an important role in the hydrolysis of polyhydroxyalkanoates [PHAs]. In the current study, Burkholderia cepacia DP1 was obtained from Penang, Malaysia in which the enzyme was purified using ion exchange and gel filtration (Superdex-75) column chromatography. The molecular mass of the enzyme was estimated to be 53.3kDa using SDS-PAGE. The enzyme activity was increased to 36.8 folds with the recovery of 16.3% after purification. The enzyme activity was detected between pH 6.0-10 and at 35-55 degrees C with pH 6.0 and 45 degrees C facilitating the maximum activity. Depolymerase was inactivated by Tween-20, Tween-80, SDS and PMSF, but insensitive to metal ions (Mg(2+), Ca(2+), K(+), Na(2+), Fe(3+)) and organic solvents (methanol, ethanol, and acetone). The apparent Km values of the purified P(3HB) depolymerase enzyme for P(3HB) and P(3HB-co-14%3HV) were 0.7mg/ml and 0.8mg/ml, respectively. The Vmax values of the purified enzyme were 10mg/min and 8.89mg/min for P(3HB) and P(3HB-co-14%3HV), respectively. The current study discovered a new extracellular poly(3-hydroxybutyrate) [P(3HB)] depolymerase enzyme from Burkholderia cepacia DP1 isolated and purified to homogeneity from the culture supernatant. To the best of our knowledge, this is the first report demonstrating the purification and biochemical characterization of P(3HB) depolymerase enzyme from genus Burkholderia.\n",
      "\n",
      "title: Extracellular polymeric substance (EPS)-degrading enzymes reduce staphylococcal surface attachment and biocide resistance on pig skin in vivo.\n",
      "authors: ['Kaplan JB', 'Mlynek KD', 'Hettiarachchi H', 'Alamneh YA', 'Biggemann L', 'Zurawski DV', 'Black CC', 'Bane CE', 'Kim RK', 'Granick MS']\n",
      "source: Staphylococcal extracellular polymeric substances (EPS) such as extracellular DNA (eDNA) and poly-N-acetylglucosamine surface polysaccharide (PNAG) mediate numerous virulence traits including host colonization and antimicrobial resistance. Previous studies showed that EPS-degrading enzymes increase staphylococcal biocide susceptibility in vitro and in vivo, and decrease virulence in animal models. In the present study we tested the effect of EPS-degrading enzymes on staphylococcal skin colonization and povidone iodine susceptibility using a novel in vivo pig model that enabled us to colonize and treat 96 isolated areas of skin on a single animal in vivo. To quantitate skin colonization, punch biopsies of colonized areas were homogenized, diluted, and plated on agar for colony forming unit enumeration. Skin was colonized with either Staphylococcus epidermidis or Staphylococcus aureus. Two EPS-degrading enzymes, DNase I and the PNAG-degrading enzyme dispersin B, were employed. Enzymes were tested for their ability to inhibit skin colonization and detach preattached bacteria. The effect of enzymes on the susceptibility of preattached S. aureus to killing by povidone iodine was also measured. We found that dispersin B significantly inhibited skin colonization by S. epidermidis and detached preattached S. epidermidis cells from skin. A cocktail of dispersin B and DNase I detached preattached S. aureus cells from skin and increased their susceptibility to killing by povidone iodine. These findings suggest that staphylococcal EPS components such as eDNA and PNAG contribute to skin colonization and biocide resistance in vivo. EPS-degrading enzymes may be a useful adjunct to conventional skin antisepsis procedures in order to further reduce skin bioburden.\n",
      "\n",
      "title: [Microbial degradation of polyurethane plastics].\n",
      "authors: ['Peng R', 'Xia M', 'Ru J', 'Huo Y', 'Yang Y']\n",
      "source: Plastic pollution has become a global environmental issue, making it necessary to explore the environmental disposal technology for plastic waste. Recently, we and other researchers have individually found microorganisms or enzymes from nature that can degrade synthetic plastic. These findings indicated that the capability of these microorganisms or enzymes to degrade plastic could be used for the disposal of plastic waste. Polyurethane (PUR) was one of the most used general plastic and its plastic waste occupied 30% of the total volume of different plastic waste. This review tried to provide a comprehensive summary of the researches on microbial degradation of PUR plastic in the past 70 years since its invention, and focused on the PUR-degrading fungi, bacteria, genes or enzymes, degradation products and the corresponding biological disposal technologies. We finally proposed the key scientific challenges on the development of high efficient biological disposal for PUR waste in the perspective researches.\n",
      "\n",
      "title: Ability of fungi isolated from plastic debris floating in the shoreline of a lake to degrade plastics.\n",
      "authors: ['Brunner I', 'Fischer M', 'Ruthi J', 'Stierli B', 'Frey B']\n",
      "source: Plastic waste in the environment is a significant threat due to its resistance to biological processes. Here we report the ability of fungal strains found on floating plastic debris to degrade plastics. In particular, we wanted to know which fungi grow on plastic debris floating in the shoreline, whether these fungi have the ability to degrade plastics, whether the plastic-degrading fungi can degrade other complex C-polymers such as lignin, and whether lignin-degraders vice versa can also break down plastics. Overall, more than a hundred fungal strains were isolated from plastic debris of the shoreline of Lake Zurich, Switzerland, and grouped morphologically. Representative strains of these groups were then selected and genetically identified, altogether twelve different fungal species and one species of Oomycota. The list of fungi included commonly occurring saprotrophic fungi but also some plant pathogens. These fungal strains were then used to test the ability to degrade polyethylene and polyurethane. The tests showed that none of the strains were able to degrade polyethylene. However, four strains were able to degrade polyurethane, the three litter-saprotrophic fungi Cladosporium cladosporioides, Xepiculopsis graminea, and Penicillium griseofulvum and the plant pathogen Leptosphaeria sp. A series of additional fungi with an origin other than from plastic debris were tested as well. Here, only the two litter-saprotrophic fungi Agaricus bisporus and Marasmius oreades showed the capability to degrade polyurethane. In contrast, wood-saprotrophic fungi and ectomycorrhizal fungi were unable to degrade polyurethane. Overall, it seems that in majority only a few litter-saprotrophic fungi, which possess a wide variety of enzymes, have the ability to degrade polyurethane. None of the fungi tested was able to degrade polyethylene.\n",
      "\n",
      "title: Synergistic Removal of Static and Dynamic Staphylococcus aureus Biofilms by Combined Treatment with a Bacteriophage Endolysin and a Polysaccharide Depolymerase.\n",
      "authors: ['Olsen NMC', 'Thiran E', 'Hasler T', 'Vanzieleghem T', 'Belibasakis GN', 'Mahillon J', 'Loessner MJ', 'Schmelcher M']\n",
      "source: Staphylococcus aureus is an important pathogen and biofilm former. Biofilms cause problems in clinics and food production and are highly recalcitrant to antibiotics and sanitizers. Bacteriophage endolysins kill bacteria by degrading their cell wall and are therefore deemed promising antimicrobials and anti-biofilm agents. Depolymerases targeting polysaccharides in the extracellular matrix have been suggested as parts of a multi-enzyme approach to eradicate biofilms. The efficacy of endolysins and depolymerases against S. aureus biofilms in static models has been demonstrated. However, there is a lack of studies evaluating their activity against biofilms grown under more realistic conditions. Here, we investigated the efficacy of the endolysin LysK and the poly-N-acetylglucosamine depolymerase DA7 against staphylococcal biofilms in static and dynamic (flow cell-based) models. LysK showed activity against multiple S. aureus strains, and both LysK and DA7 removed static and dynamic biofilms from polystyrene and glass surfaces at low micromolar and nanomolar concentrations, respectively. When combined, the enzymes acted synergistically, as demonstrated by crystal violet staining of static biofilms, significantly reducing viable cell counts compared to individual enzyme treatment in the dynamic model, and confocal laser scanning microscopy. Overall, our results suggest that LysK and DA7 are potent anti-biofilm agents, alone and in combination.\n",
      "\n",
      "title: Pretreatment with an esterase from the yeast Pseudozyma antarctica accelerates biodegradation of plastic mulch film in soil under laboratory conditions.\n",
      "authors: ['Sameshima-Yamashita Y', 'Ueda H', 'Koitabashi M', 'Kitamoto H']\n",
      "source: The yeast Pseudozyma antarctica secretes a concentrated biodegradable plastic (BP)-degrading enzyme when cultivated with xylose. Treatment with the culture filtrate reduced the puncture strength of commercial BP mulch films. After burying the film in soil, the residual amount of solid film was reduced significantly, and none was recovered after 5 weeks. The dynamics of soil fungal communities were analyzed weekly after burying the film using 18S rDNA polymerase chain reaction-denaturing gradient gel electrophoresis (PCR-DGGE) profiling of soil DNA. In the soil containing enzyme-treated film, the native community essentially recovered within 24 weeks. In comparison, the untreated solid film remained in the soil for 12 weeks and the response of the soil-fungal community was relatively slow; it had not recovered within 24 weeks.\n",
      "\n",
      "title: Phosphotriesterase-Magnetic Nanoparticle Bioconjugates with Improved Enzyme Activity in a Biocatalytic Membrane Reactor.\n",
      "authors: ['Gebreyohannes AY', 'Mazzei R', 'Marei Abdelrahim MY', 'Vitola G', 'Porzio E', 'Manco G', 'Barboiu M', 'Giorno L']\n",
      "source: The need to find alternative bioremediation solutions for organophosphate degradation pushed the research to develop technologies based on organophosphate degrading enzymes, such as phosphotriesterase. The use of free phosphotriesterase poses limits in terms of enzyme reuse, stability, and process development. The heterogenization of enzyme on a support and their use in bioreactors implemented by membranes seems a suitable strategy, thanks to the ability of membranes to compartmentalize, to govern mass transfer, and to provide a microenvironment with tuned physicochemical and structural properties. Usually, hydrophilic membranes are used since they easily guarantee the presence of water molecules needed for the enzyme catalytic activity. However, hydrophobic materials exhibit a larger shelf life and are preferred for the construction of filters and masks. Therefore, in this work, hydrophobic polyvinylidene fluoride (PVDF) porous membranes were used to develop biocatalytic membrane reactors (BMR). The phosphotriesterase-like lactonase (PLL) enzyme ( SsoPox triple mutant from S. solfataricus) endowed with thermostable phosphotriesterase activity was used as model biocatalyst. The enzyme was covalently bound directly to the PVDF hydrophobic membrane or it was bound to magnetic nanoparticles and then positioned on the hydrophobic membrane surface by means of an external magnetic field. Investigation of kinetic properties of the two BMRs and the influence of immobilized enzyme amount revealed that the performance of the BMR was mostly dependent on the amount of enzyme and its distribution on the immobilization support. Magnetic nanocomposite mediated immobilization showed a much better performance, with an observed specific activity higher than 90% compared to grafting of the enzyme on the membrane. Even though the present work focused on phosphotriesterase, it can be easily translated to other classes of enzymes and related applications.\n",
      "\n",
      "title: Characterization and engineering of a plastic-degrading aromatic polyesterase.\n",
      "authors: ['Austin HP', 'Allen MD', 'Donohoe BS', 'Rorrer NA', 'Kearns FL', 'Silveira RL', 'Pollard BC', 'Dominick G', 'Duman R', 'El Omari K', 'Mykhaylyk V', 'Wagner A', 'Michener WE', 'Amore A', 'Skaf MS', 'Crowley MF', 'Thorne AW', 'Johnson CW', 'Woodcock HL', 'McGeehan JE', 'Beckham GT']\n",
      "source: Poly(ethylene terephthalate) (PET) is one of the most abundantly produced synthetic polymers and is accumulating in the environment at a staggering rate as discarded packaging and textiles. The properties that make PET so useful also endow it with an alarming resistance to biodegradation, likely lasting centuries in the environment. Our collective reliance on PET and other plastics means that this buildup will continue unless solutions are found. Recently, a newly discovered bacterium, Ideonella sakaiensis 201-F6, was shown to exhibit the rare ability to grow on PET as a major carbon and energy source. Central to its PET biodegradation capability is a secreted PETase (PET-digesting enzyme). Here, we present a 0.92 A resolution X-ray crystal structure of PETase, which reveals features common to both cutinases and lipases. PETase retains the ancestral alpha/beta-hydrolase fold but exhibits a more open active-site cleft than homologous cutinases. By narrowing the binding cleft via mutation of two active-site residues to conserved amino acids in cutinases, we surprisingly observe improved PET degradation, suggesting that PETase is not fully optimized for crystalline PET degradation, despite presumably evolving in a PET-rich environment. Additionally, we show that PETase degrades another semiaromatic polyester, polyethylene-2,5-furandicarboxylate (PEF), which is an emerging, bioderived PET replacement with improved barrier properties. In contrast, PETase does not degrade aliphatic polyesters, suggesting that it is generally an aromatic polyesterase. These findings suggest that additional protein engineering to increase PETase performance is realistic and highlight the need for further developments of structure/activity relationships for biodegradation of synthetic polyesters.\n",
      "\n",
      "title: Macro- and micromechanical remodelling in the fish atrium is associated with regulation of collagen 1 alpha 3 chain expression.\n",
      "authors: ['Keen AN', 'Fenna AJ', 'McConnell JC', 'Sherratt MJ', 'Gardner P', 'Shiels HA']\n",
      "source: Numerous pathologies lead to remodelling of the mammalian ventricle, often associated with fibrosis. Recent work in fish has shown that fibrotic remodelling of the ventricle is 'reversible', changing seasonally as temperature-induced changes in blood viscosity alter haemodynamic load on the heart. The atrial response to varying haemodynamic load is less understood in mammals and completely unexplored in non-mammalian vertebrates. To investigate atrial remodelling, rainbow trout were chronically cooled (from 10 +/- 1 to 5 +/- 1 degrees C) and chronically warmed (from 10 +/- 1 to 18 +/- 1 degrees C) for a minimum of 8 weeks. We assessed the functional effects on compliance using ex vivo heart preparations and atomic force microscopy nano-indentation and found chronic cold increased passive stiffness of the whole atrium and micromechanical stiffness of tissue sections. We then performed histological, biochemical and molecular assays to probe the mechanisms underlying functional remodelling of the atrial tissue. We found cooling resulted in collagen deposition which was associated with an upregulation of collagen-promoting genes, including the fish-specific collagen I alpha 3 chain, and a reduction in gelatinase activity of collagen-degrading matrix metalloproteinases (MMPs). Finally, we found that cooling reduced mRNA expression of cardiac growth factors and hypertrophic markers. Following long-term warming, there was an opposing response to that seen with cooling; however, these changes were more moderate. Our findings suggest that chronic cooling causes atrial dilation and increased myocardial stiffness in trout atria analogous to pathological states defined by changes in preload or afterload of the mammalian atria. The reversal of this phenotype following chronic warming is particularly interesting as it suggests that typically pathological features of mammalian atrial remodelling may oscillate seasonally in the fish, revealing a more dynamic and plastic atrial remodelling response.\n",
      "\n",
      "title: Active Site Flexibility as a Hallmark for Efficient PET Degradation by I. sakaiensis PETase.\n",
      "authors: ['Fecker T', 'Galaz-Davison P', 'Engelberger F', 'Narui Y', 'Sotomayor M', 'Parra LP', 'Ramirez-Sarmiento CA']\n",
      "source: Polyethylene terephthalate (PET) is one of the most-consumed synthetic polymers, with an annual production of 50 million tons. Unfortunately, PET accumulates as waste and is highly resistant to biodegradation. Recently, fungal and bacterial thermophilic hydrolases were found to catalyze PET hydrolysis with optimal activities at high temperatures. Strikingly, an enzyme from Ideonella sakaiensis, termed PETase, was described to efficiently degrade PET at room temperature, but the molecular basis of its activity is not currently understood. Here, a crystal structure of PETase was determined at 2.02 A resolution and employed in molecular dynamics simulations showing that the active site of PETase has higher flexibility at room temperature than its thermophilic counterparts. This flexibility is controlled by a novel disulfide bond in its active site, with its removal leading to destabilization of the catalytic triad and reduction of the hydrolase activity. Molecular docking of a model substrate predicts that PET binds to PETase in a unique and energetically favorable conformation facilitated by several residue substitutions within its active site when compared to other enzymes. These computational predictions are in excellent agreement with recent mutagenesis and PET film degradation analyses. Finally, we rationalize the increased catalytic activity of PETase at room temperature through molecular dynamics simulations of enzyme-ligand complexes for PETase and other thermophilic PET-degrading enzymes at 298, 323, and 353 K. Our results reveal that both the binding pose and residue substitutions within PETase favor proximity between the catalytic residues and the labile carbonyl of the substrate at room temperature, suggesting a more favorable hydrolytic reaction. These results are valuable for enabling detailed evolutionary analysis of PET-degrading enzymes and for rational design endeavors aiming at increasing the efficiency of PETase and similar enzymes toward plastic degradation.\n",
      "\n",
      "title: Structural insight into molecular mechanism of poly(ethylene terephthalate) degradation.\n",
      "authors: ['Joo S', 'Cho IJ', 'Seo H', 'Son HF', 'Sagong HY', 'Shin TJ', 'Choi SY', 'Lee SY', 'Kim KJ']\n",
      "source: Plastics, including poly(ethylene terephthalate) (PET), possess many desirable characteristics and thus are widely used in daily life. However, non-biodegradability, once thought to be an advantage offered by plastics, is causing major environmental problem. Recently, a PET-degrading bacterium, Ideonella sakaiensis, was identified and suggested for possible use in degradation and/or recycling of PET. However, the molecular mechanism of PET degradation is not known. Here we report the crystal structure of I. sakaiensis PETase (IsPETase) at 1.5 A resolution. IsPETase has a Ser-His-Asp catalytic triad at its active site and contains an optimal substrate binding site to accommodate four monohydroxyethyl terephthalate (MHET) moieties of PET. Based on structural and site-directed mutagenesis experiments, the detailed process of PET degradation into MHET, terephthalic acid, and ethylene glycol is suggested. Moreover, other PETase candidates potentially having high PET-degrading activities are suggested based on phylogenetic tree analysis of 69 PETase-like proteins.\n",
      "\n",
      "title: Blocking of matrix metalloproteinases-13 responsive peptide in poly(urethane urea) for potential cartilage tissue engineering applications.\n",
      "authors: ['Wu G', 'Wang H', 'Xiao J', 'Wang L', 'Ke Y', 'Fang L', 'Deng C', 'Liao H']\n",
      "source: The matching of scaffold degradation rate with neotissue growth is required for tissue engineering applications. Timely provision of proper spaces especially for cartilage tissue engineering plays a pivotal role in chondrocyte cluster formation. In this study, poly(urethane urea) was synthesized using conventional two-stage method by extending the isocyanate group terminated prepolymers with different amounts of GPLGLWARK peptide, which responses the degrading induced by matrix metalloproteinase 13, the main proteinase for cartilage matrix degradation. The Fourier transform infrared spectrometer with the attenuated total reflection and 1H nuclear magnetic resonance spectra revealed that the peptides were introduced to poly(urethane urea) according to the characteristic absorption bands of the peptide and the newly formed urea bonds. The ultraviolet-visible spectroscopy spectra showed that the weight percentages of the peptide in the three poly(urethane urea) were 25%, 32%, and 35%. Atomic force microscopy images revealed that phase separation occurred in all poly(urethane urea) samples and became increasingly apparent with increasing amount of peptides introduced. Mechanical tests showed that the poly(urethane urea) strength increased with increasing amount of peptides in poly(urethane urea). Poly(urethane urea) proteolysis in matrix metalloproteinase 13 solution was more rapid than hydrolysis in aqueous buffer, and proteolysis rate was dependent on the amount of peptides in poly(urethane urea). Cell proliferation on the material surface in vitro displayed nontoxicity for all synthesized poly(urethane urea). In vivo subcutaneous implantation evaluation revealed the presence of local foreign body reactions triggered by poly(urethane urea) but was not due to peptide in poly(urethane urea). Moreover, the synthesized poly(urethane urea) with significant phase separation did not degrade under the matrix metalloproteinase 13 free subcutaneous environment, but poly(urethane urea) with minimal phase separation was degraded by attacking of the enzymes adsorbed on the hydrophobic surface through non-specific adsorption.\n",
      "\n",
      "title: The role of NFkappaB in spheroid formation of human breast cancer cells cultured on the Random Positioning Machine.\n",
      "authors: ['Kopp S', 'Sahana J', 'Islam T', 'Petersen AG', 'Bauer J', 'Corydon TJ', 'Schulz H', 'Saar K', 'Huebner N', 'Slumstrup L', 'Riwaldt S', 'Wehland M', 'Infanger M', 'Luetzenberg R', 'Grimm D']\n",
      "source: Human MCF-7 breast cancer cells were exposed to a Random Positioning Machine (RPM). After 24 hours (h) the cells grew either adherently within a monolayer (AD) or within multicellular spheroids (MCS). AD and MCS populations were separately harvested, their cellular differences were determined performing qPCR on genes, which were differently expressed in AD and MCS cells. Gene array technology was applied to detect RPM-sensitive genes in MCF-7 cells after 24 h. Furthermore, the capability to form multicellular spheroids in vitro was compared with the intracellular distribution of NF-kappaB (NFkappaB) p65. NFkappaB was equally distributed in static control cells, but predominantly localized in the cytoplasm in AD cells and nucleus in MCS cells exposed to the RPM. Gene array analyses revealed a more than 2-fold change of only 23 genes including some whose products are affected by oxygen levels or regulate glycolysis. Significant upregulations of the mRNAs of enzymes degrading heme, of ANXA1, ANXA2, CTGF, CAV2 and ICAM1, as well as of FAS, Casp8, BAX, p53, CYC1 and PARP1 were observed in MCS cells as compared with 1g-control and AD cells. An interaction analysis of 47 investigated genes suggested that HMOX-1 and NFkappaB variants are activated, when multicellular spheroids are formed.\n",
      "\n",
      "title: Metabolic pathway of 6-aminohexanoate in the nylon oligomer-degrading bacterium Arthrobacter sp. KI72: identification of the enzymes responsible for the conversion of 6-aminohexanoate to adipate.\n",
      "authors: ['Takehara I', 'Fujii T', 'Tanimoto Y', 'Kato DI', 'Takeo M', 'Negoro S']\n",
      "source: Arthrobacter sp. strain KI72 grows on a 6-aminohexanoate oligomer, which is a by-product of nylon-6 manufacturing, as a sole source of carbon and nitrogen. We cloned the two genes, nylD 1 and nylE 1 , responsible for 6-aminohexanoate metabolism on the basis of the draft genomic DNA sequence of strain KI72. We amplified the DNA fragments that encode these genes by polymerase chain reaction using a synthetic primer DNA homologous to the 4-aminobutyrate metabolic enzymes. We inserted the amplified DNA fragments into the expression vector pColdI in Escherichia coli, purified the His-tagged enzymes to homogeneity, and performed biochemical studies. We confirmed that 6-aminohexanoate aminotransferase (NylD1) catalyzes the reaction of 6-aminohexanoate to adipate semialdehyde using alpha-ketoglutarate, pyruvate, and glyoxylate as amino acceptors, generating glutamate, alanine, and glycine, respectively. The reaction requires pyridoxal phosphate (PLP) as a cofactor. For further metabolism, adipate semialdehyde dehydrogenase (NylE1) catalyzes the oxidative reaction of adipate semialdehyde to adipate using NADP(+) as a cofactor. Phylogenic analysis revealed that NylD1 should be placed in a branch of the PLP-dependent aminotransferase sub III, while NylE1 should be in a branch of the aldehyde dehydrogenase superfamily. In addition, we established a NylD1/NylE1 coupled system to quantify the aminotransferase activity and to enable the conversion of 6-aminohexaoate to adipate via adipate semialdehyde with a yield of > 90%. In the present study, we demonstrate that 6-aminohexanoate produced from polymeric nylon-6 and nylon oligomers (i.e., a mixture of 6-aminohexaoate oligomers) by nylon hydrolase (NylC) and 6-aminohexanoate dimer hydrolase (NylB) reactions are sequentially converted to adipate by metabolic engineering technology.\n",
      "\n"
     ]
    }
   ],
   "source": [
    "from Bio import Entrez, Medline, SeqIO\n",
    "Entrez.email = \"astuart@ramapo.edu\" \n",
    "\n",
    "\n",
    "handle = Entrez.esearch(db=\"pubmed\", term=\"plastic AND degrading AND enzymes\", retmax=50)\n",
    "record = Entrez.read(handle)\n",
    "idlist = record[\"IdList\"]\n",
    "\n",
    "print(\"Records Found:\",record[\"Count\"])\n",
    "\n",
    "print(\"50 Id's for PubMed IDs of articles related to plastic and enzymes\\n {}\".format(idlist[:50]))\n",
    "\n",
    "handle = Entrez.efetch(db=\"pubmed\", id=idlist, rettype=\"medline\")\n",
    "records = Medline.parse(handle)\n",
    "records = list(records)\n",
    "\n",
    "for record in records:\n",
    "    print(\"title:\", record.get(\"TI\", \"?\"))\n",
    "    print(\"authors:\", record.get(\"AU\", \"?\"))\n",
    "    print(\"source:\", record.get(\"AB\", \"?\"))\n",
    "    print(\"\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "The WebEnv is NCID_1_80524764_130.14.18.97_9001_1580781328_1819903118_0MetA0_S_MegaStore\n",
      "The QueryKey is 1\n",
      "Data Type of search_results  <class 'Bio.Entrez.Parser.DictionaryElement'>\n",
      "dict_keys(['Count', 'RetMax', 'RetStart', 'QueryKey', 'WebEnv', 'IdList', 'TranslationSet', 'TranslationStack', 'QueryTranslation', 'ErrorList'])\n",
      "Results found = 200\n",
      "Found 200 results\n",
      "Query Translation = (\"saccharomyces\"[MeSH Terms] OR \"Saccharomyces\"[Organism] OR Saccharomyces[All Fields]) AND (\"beer\"[MeSH Terms] OR beer[All Fields])\n",
      "['200133146', '200077969', '200035549', '200069723', '200052380', '200052379', '200052378', '200045951', '200045950', '200045947']\n",
      "Going to download record 1 to 10\n",
      "\n",
      "\n",
      "1. Maltotriose consumption by hybrid Saccharomyces pastorianus is heterotic and results from regulatory cross-talk between parental sub-genomes\n",
      "\n",
      "(Submitter supplied) S. pastorianus strains are hybrids of S. cerevisiae and S. eubayanus that have been domesticated for several centuries in lager-beer brewing environments. As sequences and structures of S. pastorianus genomes are being resolved, molecular mechanisms and evolutionary origin of several industrially relevant phenotypes remain unknown. This study investigates how maltotriose metabolism, a key feature in brewing, may have arisen in early S. more...\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae x Saccharomyces eubayanus; Saccharomyces eubayanus\n",
      "\n",
      "Type:\t\tExpression profiling by high throughput sequencing\n",
      "\n",
      "Platforms: GPL26822 GPL25341 20 Samples\n",
      "\n",
      "FTP download: GEO (CSV) ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE133nnn/GSE133146/\n",
      "\n",
      "SRA Run Selector: https://www.ncbi.nlm.nih.gov/Traces/study/?acc=PRJNA550094\n",
      "\n",
      "Series\t\tAccession: GSE133146\tID: 200133146\n",
      "\n",
      "\n",
      "\n",
      "2. Gene expression profiles of human immature dendritic cells after 12h of co-cultivation with Aspergillus fumigatus, Candida albicans and LPS\n",
      "\n",
      "(Submitter supplied) In a whole-transcriptome study, cellular responses of DCs confronted with the fungi A. fumigatus, C. albicans or the bacterial cell wall component lipopolysaccharide (LPS) were investigated. Therefore, DCs of four independent donors were harvested after 12 hours co-culture with A. fumigatus, C. albicans and LPS and analyzed with Affymetrix whole genome expression arrays. In general, transcriptomic analysis revealed a clustering of the A. more...\n",
      "\n",
      "Organism:\tHomo sapiens\n",
      "\n",
      "Type:\t\tExpression profiling by array\n",
      "\n",
      "Platform: GPL13667 15 Samples\n",
      "\n",
      "FTP download: GEO (CEL, TXT) ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE77nnn/GSE77969/\n",
      "\n",
      "Series\t\tAccession: GSE77969\tID: 200077969\n",
      "\n",
      "\n",
      "\n",
      "3. Array CGH of interspecific hybrid yeasts using multi-species array platform\n",
      "\n",
      "(Submitter supplied) Four hybrid yeast strains isolated from a variety of industrial substrates were hybridized to an array-CGH platform containing probes to query the whole genomes of seven different Saccharomyces species.  For most of the strains we found evidence of multiple interspecific hybridization events and multiple introgressed regions.  The strains queried were GSY205 (isolated from a cider fermentation), GSY505 (a contaminant from a lager beer fermentation), GSY2232 (a commercial wine yeast strain), and GSY312 (a commercial lager beer strain). more...\n",
      "\n",
      "Organism:\tSaccharomyces bayanus; Saccharomyces cerevisiae\n",
      "\n",
      "Type:\t\tGenome variation profiling by array\n",
      "\n",
      "Platforms: GPL11575 GPL9174 GPL9175 119 Samples\n",
      "\n",
      "FTP download: GEO ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE35nnn/GSE35549/\n",
      "\n",
      "Series\t\tAccession: GSE35549\tID: 200035549\n",
      "\n",
      "\n",
      "\n",
      "4. Gene expression profiles of human immature dendritic cells co-cultivated with Aspergillus fumigatus, Candida albicans and LPS for 6h\n",
      "\n",
      "(Submitter supplied) In a whole-transcriptome study, cellular responses of DCs confronted with the fungi A. fumigatus, C. albicans or the bacterial cell wall component LPS were investigated. Therefore DCs of four independent donors were analyzed after 6  hours co-culture with A. fumigatus, C. albicans and LPS by Affymetrix whole genome expression arrays. In general, transcriptomic analysis revealed a clustering of the A. more...\n",
      "\n",
      "Organism:\tHomo sapiens\n",
      "\n",
      "Type:\t\tExpression profiling by array\n",
      "\n",
      "Platform: GPL13667 16 Samples\n",
      "\n",
      "FTP download: GEO (CEL) ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE69nnn/GSE69723/\n",
      "\n",
      "Series\t\tAccession: GSE69723\tID: 200069723\n",
      "\n",
      "\n",
      "\n",
      "5. Comparison between brewery A and brewery B\n",
      "\n",
      "(Submitter supplied) This SuperSeries is composed of the SubSeries listed below.\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae; Saccharomyces pastorianus\n",
      "\n",
      "Type:\t\tExpression profiling by array\n",
      "\n",
      "Platform: GPL17932 14 Samples\n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE52nnn/GSE52380/\n",
      "\n",
      "Series\t\tAccession: GSE52380\tID: 200052380\n",
      "\n",
      "\n",
      "\n",
      "6. Comparison between all runs of a serial repitching in brewery B\n",
      "\n",
      "(Submitter supplied) The aim of this study was to identify differently expressed genes between all 6 runs of brewery B. A run describes the industrial fermentation of brewery wort. By the end of fermentation the yeast slurry is cropped. These cells are reused for inoculation of another cylindro-conical vessel (CCVs).\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae; Saccharomyces pastorianus\n",
      "\n",
      "Type:\t\tExpression profiling by array\n",
      "\n",
      "Platform: GPL17932 7 Samples\n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE52nnn/GSE52379/\n",
      "\n",
      "Series\t\tAccession: GSE52379\tID: 200052379\n",
      "\n",
      "\n",
      "\n",
      "7. Comparison between all runs of a serial repitching in brewery A\n",
      "\n",
      "(Submitter supplied) The aim of this study was to identify differently expressed genes between all 6 runs of brewery A. A run describes the industrial fermentation of brewery wort. By the end of fermentation the yeast slurry is cropped. These cells are reused for inoculation of another cylindro-conical vessel (CCVs).\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae; Saccharomyces pastorianus\n",
      "\n",
      "Type:\t\tExpression profiling by array\n",
      "\n",
      "Platform: GPL17932 7 Samples\n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE52nnn/GSE52378/\n",
      "\n",
      "Series\t\tAccession: GSE52378\tID: 200052378\n",
      "\n",
      "\n",
      "\n",
      "8. Serial industrial repitching and stress\n",
      "\n",
      "(Submitter supplied) This SuperSeries is composed of the SubSeries listed below.\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Type:\t\tExpression profiling by array\n",
      "\n",
      "Platforms: GPL2603 GPL16996 88 Samples\n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE45nnn/GSE45951/\n",
      "\n",
      "Series\t\tAccession: GSE45951\tID: 200045951\n",
      "\n",
      "\n",
      "\n",
      "9. Comparison between a first and a last run of industrial serial repitching\n",
      "\n",
      "(Submitter supplied) The aim of this study was to identify differently expressed genes between first and last runs of serial repitchings from four different German breweries. In the breweries the number of successive runs varied from 4 to 22. A run describes the industrial fermentation of brewery wort. By the end of fermentation the yeast slurry is cropped. These cells are reused for inoculation of another cylindro-conical vessel (CCVs).\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Type:\t\tExpression profiling by array\n",
      "\n",
      "Platform: GPL2603 8 Samples\n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE45nnn/GSE45950/\n",
      "\n",
      "Series\t\tAccession: GSE45950\tID: 200045950\n",
      "\n",
      "\n",
      "\n",
      "10. Comparison between all runs of industrial serial repitching 4\n",
      "\n",
      "(Submitter supplied) The aim of this study was to identify differently expressed genes between all runs of four serial repitchings from a German brewery. In the brewery the number of successive runs varied from 17 to 20. A run describes the industrial fermentation of brewery wort. By the end of fermentation the yeast slurry is cropped. These cells are reused for inoculation of another cylindro-conical vessel (CCVs).\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Type:\t\tExpression profiling by array\n",
      "\n",
      "Platform: GPL16996 20 Samples\n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE45nnn/GSE45947/\n",
      "\n",
      "Series\t\tAccession: GSE45947\tID: 200045947\n",
      "\n",
      "Going to download record 11 to 20\n",
      "\n",
      "\n",
      "1. Comparison between all runs of industrial serial repitching 3\n",
      "\n",
      "(Submitter supplied) The aim of this study was to identify differently expressed genes between all runs of four serial repitchings from a German brewery. In the brewery the number of successive runs varied from 17 to 20. A run describes the industrial fermentation of brewery wort. By the end of fermentation the yeast slurry is cropped. These cells are reused for inoculation of another cylindro-conical vessel (CCVs).\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Type:\t\tExpression profiling by array\n",
      "\n",
      "Platform: GPL16996 20 Samples\n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE45nnn/GSE45946/\n",
      "\n",
      "Series\t\tAccession: GSE45946\tID: 200045946\n",
      "\n",
      "\n",
      "\n",
      "2. Comparison between all runs of industrial serial repitching 2\n",
      "\n",
      "(Submitter supplied) The aim of this study was to identify differently expressed genes between all runs of four serial repitchings from a German brewery. In the brewery the number of successive runs varied from 17 to 20. A run describes the industrial fermentation of brewery wort. By the end of fermentation the yeast slurry is cropped. These cells are reused for inoculation of another cylindro-conical vessel (CCVs).\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Type:\t\tExpression profiling by array\n",
      "\n",
      "Platform: GPL16996 17 Samples\n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE45nnn/GSE45944/\n",
      "\n",
      "Series\t\tAccession: GSE45944\tID: 200045944\n",
      "\n",
      "\n",
      "\n",
      "3. Comparison between all runs of industrial serial repitching 1\n",
      "\n",
      "(Submitter supplied) The aim of this study was to identify differently expressed genes between all runs of four serial repitchings from a German brewery. In the brewery the number of successive runs varied from 17 to 20. A run describes the industrial fermentation of brewery wort. By the end of fermentation the yeast slurry is cropped. These cells are reused for inoculation of another cylindro-conical vessel (CCVs).\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Type:\t\tExpression profiling by array\n",
      "\n",
      "Platform: GPL16996 18 Samples\n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE45nnn/GSE45943/\n",
      "\n",
      "Series\t\tAccession: GSE45943\tID: 200045943\n",
      "\n",
      "\n",
      "\n",
      "4. A comparative encyclopedia of DNA elements in the mouse genome\n",
      "\n",
      "(Submitter supplied) The laboratory mouse shares the majority of its protein-coding genes with humans, making it the premier model organism in biomedical research, yet the two mammals differ in significant ways. To gain greater insights into both shared and species-specific transcriptional and cellular regulatory programs in the mouse, the Mouse ENCODE Consortium has mapped transcription, DNase I hypersensitivity, transcription factor binding, chromatin modifications and replication domains throughout the mouse genome in diverse cell and tissue types. more...\n",
      "\n",
      "Organism:\tMus musculus\n",
      "\n",
      "Type:\t\tExpression profiling by high throughput sequencing; Genome variation profiling by genome tiling array; Genome binding/occupancy profiling by high throughput sequencing\n",
      "\n",
      "8 related Platforms 588 Samples\n",
      "\n",
      "FTP download: GEO (BAM, BEDRNAELEMENTS, BIGWIG, BROADPEAK, NARROWPEAK, PAIR) ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE49nnn/GSE49847/\n",
      "\n",
      "Series\t\tAccession: GSE49847\tID: 200049847\n",
      "\n",
      "\n",
      "\n",
      "5. APJ1 and GRE3 Homologs Work in Concert to Allow Growth in Xylose in a Genetically Intractable Natural Saccharomyces sensu stricto Hybrid Yeast\n",
      "\n",
      "(Submitter supplied) Creating Saccharomyces yeasts capable of efficient fermentation of pentoses such as xylose remains a key challenge in the production of ethanol from lignocellulosic biomass.  Metabolic engineering of industrial Saccharomyces cerevisiae strains has yielded xylose-fermenting strains, but these strains have not yet achieved industrial viability due largely to xylose fermentation being prohibitively slower than that of glucose. more...\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Type:\t\tGenome variation profiling by array\n",
      "\n",
      "Platform: GPL7077 4 Samples\n",
      "\n",
      "FTP download: GEO ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE35nnn/GSE35528/\n",
      "\n",
      "Series\t\tAccession: GSE35528\tID: 200035528\n",
      "\n",
      "\n",
      "\n",
      "6. Co-inoculation with Dekkerea bruxellensis impacts the trancriptome of Saccharomyces cerevisiae during wine fermentation\n",
      "\n",
      "(Submitter supplied) The yeast Dekkera bruxellensis is as ethanol tolerant as Saccharomyces cerevisiae and may be found in bottled wine. It causes the spoilage of wine, beer, cider and soft drinks. In wines, the metabolic products responsible for spoilage by Dekkera bruxellensis are mainly volatile phenols. These chemical compounds are responsible for the taints described as medicinal in white wines (due to vinyl phenols) and as leather, horse sweat and stable in red wines (due to ethyl phenols mainly 4-ethylphenol). more...\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Type:\t\tExpression profiling by array\n",
      "\n",
      "Platform: GPL11488 16 Samples\n",
      "\n",
      "FTP download: GEO (TXT) ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE38nnn/GSE38582/\n",
      "\n",
      "Series\t\tAccession: GSE38582\tID: 200038582\n",
      "\n",
      "\n",
      "\n",
      "7. Revealing the beneficial effect of protease supplementation to high gravity beer fermentations  using omics techniques\n",
      "\n",
      "(Submitter supplied) Addition of sugar syrups to the basic wort is a popular technique to achieve higher gravity in beer fermentations, but it results in dilution of the free amino nitrogen (FAN) content in the medium. The multicomponent protease enzyme Flavourzyme has beneficial effect on the brewers yeast fermentation performance during high gravity fermentations as it increases the initial FAN value and results in higher FAN uptake, higher specific growth rate, higher ethanol yield and improved flavour profile. more...\n",
      "\n",
      "Organism:\tSchizosaccharomyces pombe; Saccharomyces cerevisiae; Saccharomyces pastorianus Weihenstephan 34/70\n",
      "\n",
      "Type:\t\tExpression profiling by array\n",
      "\n",
      "Platform: GPL2529 7 Samples\n",
      "\n",
      "FTP download: GEO (CEL) ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE24nnn/GSE24645/\n",
      "\n",
      "Series\t\tAccession: GSE24645\tID: 200024645\n",
      "\n",
      "\n",
      "\n",
      "8. Karyotyping of maltose-fermenting Saccharomyces yeasts with differing maltotriose utilization profiles\n",
      "\n",
      "(Submitter supplied) Aims: We performed an analysis of maltotriose utilization by 52 Saccharomyces yeast strains able to ferment maltose efficiently and correlated the observed phenotypes with differences in the copy number of genes possibly involved in maltotriose utilization by yeast cells. Methods and Results: The analysis of maltose and maltotriose utilization by laboratory and industrial strains of the species Saccharomyces cerevisiae and Saccharomyces pastorianus (a natural S. more...\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Type:\t\tGenome variation profiling by array\n",
      "\n",
      "Platforms: GPL2634 GPL55 GPL10715 13 Samples\n",
      "\n",
      "FTP download: GEO ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE23nnn/GSE23055/\n",
      "\n",
      "Series\t\tAccession: GSE23055\tID: 200023055\n",
      "\n",
      "\n",
      "\n",
      "9. Fuel Ethanol Yeast Strains\n",
      "\n",
      "(Submitter supplied) Fuel ethanol is now considered a global energy commodity that is fully competitive with gasoline.  We have determined genome copy number differences that are common to five industrially important fuel ethanol yeast strains responsible for the production of billions of gallons of fuel ethanol per year from sugarcane.  The fuel strains used were CAT1, BG1, PE2, SA1, and VR1 (note that two independent isolates were analyzed, denoted by \"-1\" and \"-2\"). more...\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Type:\t\tGenome variation profiling by array\n",
      "\n",
      "4 related Platforms 23 Samples\n",
      "\n",
      "FTP download: GEO ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE13nnn/GSE13875/\n",
      "\n",
      "Series\t\tAccession: GSE13875\tID: 200013875\n",
      "\n",
      "\n",
      "\n",
      "10. Saccharomyces pastorianus strains comparison\n",
      "\n",
      "(Submitter supplied) Saccharomyces pastorianus is the yeast used to make lager beer; it is known to be an interspecific hybrid formed by the fusion between S. cerevisiae and S. bayanus genomes.  This data set queries 17 S. pastorianus strains, collected at various times over the last 125 years from various breweries located in different geographical locations, which were obtained from CBS and DBVPG culture collections.  The data in this set represent array-CGH experiments performed with these strains, using \"2-species\" custom Agilent arrays (the \"2-species\" arrays contain probes spaced every ~2 kb across the whole genomes of both S. cerevisiae and S. bayanus; the probes are unique and specific for each genome).  The data set also contains 3 self-self hybridizations (S. cerevisiae + S. bayanus DNA mixed together in equimolar amounts, then labeled green or red in separate reactions, then hybridized to the \"2-species\" arrays) used for normalization in CGH-Miner analysis. A strain or line experiment design type assays differences between multiple strains, cultivars, serovars, isolates, lines from organisms of a single species.\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Type:\t\tGenome variation profiling by genome tiling array\n",
      "\n",
      "Platform: GPL7077 25 Samples\n",
      "\n",
      "FTP download: GEO ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE12nnn/GSE12177/\n",
      "\n",
      "Series\t\tAccession: GSE12177\tID: 200012177\n",
      "\n",
      "Going to download record 21 to 30\n",
      "\n",
      "\n",
      "1. Trasncriptional response of Saccharomyces cerevisiae to nitrogen limitation in chemostat culture\n",
      "\n",
      "(Submitter supplied) Zinc is indispensable for the catalytic activity and structural stability of many proteins, and its deficiency can have severe consequences for microbial growth in natural and industrial environments. For example, Zn depletion in wort negatively affects beer fermentation and quality. Several studies have investigated yeast adaptation to low Zn supply, but were all performed in batch cultures, where specific growth rate depends on Zn availability. more...\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Type:\t\tExpression profiling by array\n",
      "\n",
      "Platform: GPL90 6 Samples\n",
      "\n",
      "FTP download: GEO (CEL, CHP, EXP) ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE8nnn/GSE8089/\n",
      "\n",
      "Series\t\tAccession: GSE8089\tID: 200008089\n",
      "\n",
      "\n",
      "\n",
      "2. Transcriptional responses of Saccharomyces cerevisiae to carbon limitation in aerobic chemostat cultures\n",
      "\n",
      "(Submitter supplied) Zinc is indispensable for the catalytic activity and structural stability of many proteins, and its deficiency can have severe consequences for microbial growth in natural and industrial environments. For example, Zn depletion in wort negatively affects beer fermentation and quality. Several studies have investigated yeast adaptation to low Zn supply, but were all performed in batch cultures, where specific growth rate depends on Zn availability. more...\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Type:\t\tExpression profiling by array\n",
      "\n",
      "Platform: GPL90 3 Samples\n",
      "\n",
      "FTP download: GEO (CEL, CHP, EXP) ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE8nnn/GSE8088/\n",
      "\n",
      "Series\t\tAccession: GSE8088\tID: 200008088\n",
      "\n",
      "\n",
      "\n",
      "3. Physiological and transcriptional responses of Saccharomyces cerevisiae to zinc limitation in chemostat cultures\n",
      "\n",
      "(Submitter supplied) Zinc is indispensable for the catalytic activity and structural stability of many proteins, and its deficiency can have severe consequences for microbial growth in natural and industrial environments. For example, Zn depletion in wort negatively affects beer fermentation and quality. Several studies have investigated yeast adaptation to low Zn supply, but were all performed in batch cultures, where specific growth rate depends on Zn availability. more...\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Type:\t\tExpression profiling by array\n",
      "\n",
      "Platform: GPL90 6 Samples\n",
      "\n",
      "FTP download: GEO (CEL, CHP, EXP) ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE8nnn/GSE8035/\n",
      "\n",
      "Series\t\tAccession: GSE8035\tID: 200008035\n",
      "\n",
      "\n",
      "\n",
      "4. Oligonucleotide microarrays for the detection and identification of viable beer spoilage bacteria\n",
      "\n",
      "(Submitter supplied) A prototype oligonucleotide microarray was designed to detect and identify viable bacterial species with the potential to grow of common beer spoilage microorganisms from the genera Lactobacillus, Megasphaera, Pediococcus and Pectinatus. Probes targeted the intergenic spacer regions (ISR) between 16S and 23S rRNA, which were amplified in a combination of reverse transcriptase (RT) and polymerase chain reaction (PCR) prior to hybridization. more...\n",
      "\n",
      "Organism:\tPediococcus inopinatus; Lactobacillus brevis; Megasphaera cerevisiae; Lactobacillus casei; Pectinatus cerevisiiphilus; Pectinatus frisingensis; Lactobacillus coryniformis; Pediococcus damnosus; Lactobacillus perolens\n",
      "\n",
      "Type:\t\tExpression profiling by array\n",
      "\n",
      "Platforms: GPL5157 GPL5158 12 Samples\n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE7nnn/GSE7840/\n",
      "\n",
      "Series\t\tAccession: GSE7840\tID: 200007840\n",
      "\n",
      "\n",
      "\n",
      "5. Lager beer fermentation time course\n",
      "\n",
      "(Submitter supplied) The dynamics of the Saccharomyces carlsbergensis brewing yeast transcriptome during a production scale lager beer fermentation.   The transcriptome of a lager brewing yeast (Saccharomyces carlsbergensis, syn. of S. pastorianus), was analysed at 12 different time points spanning a production-scale lager beer fermentation. Generally, the average expression rapidly increased and had a maximum value on day 2, then decreased as the sugar got consumed. more...\n",
      "\n",
      "Organism:\tSaccharomyces pastorianus; Saccharomyces cerevisiae\n",
      "\n",
      "Type:\t\tExpression profiling by array\n",
      "\n",
      "Platform: GPL205 12 Samples\n",
      "\n",
      "FTP download: GEO ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSEnnn/GSE79/\n",
      "\n",
      "Series\t\tAccession: GSE79\tID: 200000079\n",
      "\n",
      "\n",
      "\n",
      "6. S4 Af 12h\n",
      "\n",
      "Organism:\tHomo sapiens\n",
      "\n",
      "Source name:\tDC 12h + A. fumigatus germ tubes\n",
      "\n",
      "Platform: GPL13667 Series: GSE77969 \n",
      "\n",
      "FTP download: GEO (CEL) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2062nnn/GSM2062993/\n",
      "\n",
      "Sample\t\tAccession: GSM2062993\tID: 302062993\n",
      "\n",
      "\n",
      "\n",
      "7. S4 Ca 12h\n",
      "\n",
      "Organism:\tHomo sapiens\n",
      "\n",
      "Source name:\tDC 12h + C. albicans wild type\n",
      "\n",
      "Platform: GPL13667 Series: GSE77969 \n",
      "\n",
      "FTP download: GEO (CEL) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2062nnn/GSM2062992/\n",
      "\n",
      "Sample\t\tAccession: GSM2062992\tID: 302062992\n",
      "\n",
      "\n",
      "\n",
      "8. S4 LPS 12h\n",
      "\n",
      "Organism:\tHomo sapiens\n",
      "\n",
      "Source name:\tDC 12h + LPS\n",
      "\n",
      "Platform: GPL13667 Series: GSE77969 \n",
      "\n",
      "FTP download: GEO (CEL) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2062nnn/GSM2062991/\n",
      "\n",
      "Sample\t\tAccession: GSM2062991\tID: 302062991\n",
      "\n",
      "\n",
      "\n",
      "9. S4 Control 12h\n",
      "\n",
      "Organism:\tHomo sapiens\n",
      "\n",
      "Source name:\tDC 12h negative control\n",
      "\n",
      "Platform: GPL13667 Series: GSE77969 \n",
      "\n",
      "FTP download: GEO (CEL) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2062nnn/GSM2062990/\n",
      "\n",
      "Sample\t\tAccession: GSM2062990\tID: 302062990\n",
      "\n",
      "\n",
      "\n",
      "10. S3 Af 12h\n",
      "\n",
      "Organism:\tHomo sapiens\n",
      "\n",
      "Source name:\tDC 12h + A. fumigatus germ tubes\n",
      "\n",
      "Platform: GPL13667 Series: GSE77969 \n",
      "\n",
      "FTP download: GEO (CEL) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2062nnn/GSM2062989/\n",
      "\n",
      "Sample\t\tAccession: GSM2062989\tID: 302062989\n",
      "\n",
      "Going to download record 31 to 40\n",
      "\n",
      "\n",
      "1. S3 LPS 12h\n",
      "\n",
      "Organism:\tHomo sapiens\n",
      "\n",
      "Source name:\tDC 12h + LPS\n",
      "\n",
      "Platform: GPL13667 Series: GSE77969 \n",
      "\n",
      "FTP download: GEO (CEL) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2062nnn/GSM2062988/\n",
      "\n",
      "Sample\t\tAccession: GSM2062988\tID: 302062988\n",
      "\n",
      "\n",
      "\n",
      "2. S3 Control 12h\n",
      "\n",
      "Organism:\tHomo sapiens\n",
      "\n",
      "Source name:\tDC 12h negative control\n",
      "\n",
      "Platform: GPL13667 Series: GSE77969 \n",
      "\n",
      "FTP download: GEO (CEL) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2062nnn/GSM2062987/\n",
      "\n",
      "Sample\t\tAccession: GSM2062987\tID: 302062987\n",
      "\n",
      "\n",
      "\n",
      "3. S2 Af 12h\n",
      "\n",
      "Organism:\tHomo sapiens\n",
      "\n",
      "Source name:\tDC 12h + A. fumigatus germ tubes\n",
      "\n",
      "Platform: GPL13667 Series: GSE77969 \n",
      "\n",
      "FTP download: GEO (CEL) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2062nnn/GSM2062986/\n",
      "\n",
      "Sample\t\tAccession: GSM2062986\tID: 302062986\n",
      "\n",
      "\n",
      "\n",
      "4. S2 Ca 12h\n",
      "\n",
      "Organism:\tHomo sapiens\n",
      "\n",
      "Source name:\tDC 12h + C. albicans wild type\n",
      "\n",
      "Platform: GPL13667 Series: GSE77969 \n",
      "\n",
      "FTP download: GEO (CEL) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2062nnn/GSM2062985/\n",
      "\n",
      "Sample\t\tAccession: GSM2062985\tID: 302062985\n",
      "\n",
      "\n",
      "\n",
      "5. S2 LPS 12h\n",
      "\n",
      "Organism:\tHomo sapiens\n",
      "\n",
      "Source name:\tDC 12h + LPS\n",
      "\n",
      "Platform: GPL13667 Series: GSE77969 \n",
      "\n",
      "FTP download: GEO (CEL) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2062nnn/GSM2062984/\n",
      "\n",
      "Sample\t\tAccession: GSM2062984\tID: 302062984\n",
      "\n",
      "\n",
      "\n",
      "6. S2 Control 12h\n",
      "\n",
      "Organism:\tHomo sapiens\n",
      "\n",
      "Source name:\tDC 12h negative control\n",
      "\n",
      "Platform: GPL13667 Series: GSE77969 \n",
      "\n",
      "FTP download: GEO (CEL) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2062nnn/GSM2062983/\n",
      "\n",
      "Sample\t\tAccession: GSM2062983\tID: 302062983\n",
      "\n",
      "\n",
      "\n",
      "7. S1 Af 12h\n",
      "\n",
      "Organism:\tHomo sapiens\n",
      "\n",
      "Source name:\tDC 12h + A. fumigatus germ tubes\n",
      "\n",
      "Platform: GPL13667 Series: GSE77969 \n",
      "\n",
      "FTP download: GEO (CEL) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2062nnn/GSM2062982/\n",
      "\n",
      "Sample\t\tAccession: GSM2062982\tID: 302062982\n",
      "\n",
      "\n",
      "\n",
      "8. S1 Ca 12h\n",
      "\n",
      "Organism:\tHomo sapiens\n",
      "\n",
      "Source name:\tDC 12h + C. albicans wild type\n",
      "\n",
      "Platform: GPL13667 Series: GSE77969 \n",
      "\n",
      "FTP download: GEO (CEL) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2062nnn/GSM2062981/\n",
      "\n",
      "Sample\t\tAccession: GSM2062981\tID: 302062981\n",
      "\n",
      "\n",
      "\n",
      "9. S1 LPS 12h\n",
      "\n",
      "Organism:\tHomo sapiens\n",
      "\n",
      "Source name:\tDC 12h + LPS\n",
      "\n",
      "Platform: GPL13667 Series: GSE77969 \n",
      "\n",
      "FTP download: GEO (CEL) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2062nnn/GSM2062980/\n",
      "\n",
      "Sample\t\tAccession: GSM2062980\tID: 302062980\n",
      "\n",
      "\n",
      "\n",
      "10. S1 Control 12h\n",
      "\n",
      "Organism:\tHomo sapiens\n",
      "\n",
      "Source name:\tDC 12h negative control\n",
      "\n",
      "Platform: GPL13667 Series: GSE77969 \n",
      "\n",
      "FTP download: GEO (CEL) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2062nnn/GSM2062979/\n",
      "\n",
      "Sample\t\tAccession: GSM2062979\tID: 302062979\n",
      "\n",
      "Going to download record 41 to 50\n",
      "\n",
      "\n",
      "1. v2 aCGH GSY312\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\t165 ng sonicated complete S. cerevisiae DNA pool plus 975 ng complete other spp DNA pool (channel 1) 165 ng HaeIII-cut genomic DNA of GSY312 (channel 2)\n",
      "\n",
      "Platform: GPL11575 Series: GSE35549 \n",
      "\n",
      "Sample\t\tAccession: GSM657024\tID: 300657024\n",
      "\n",
      "\n",
      "\n",
      "2. DC4 6h Af GT\n",
      "\n",
      "Organism:\tHomo sapiens\n",
      "\n",
      "Source name:\tDC 6h + A. fumigatus germ tubes\n",
      "\n",
      "Platform: GPL13667 Series: GSE69723 \n",
      "\n",
      "FTP download: GEO (CEL) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1707nnn/GSM1707897/\n",
      "\n",
      "Sample\t\tAccession: GSM1707897\tID: 301707897\n",
      "\n",
      "\n",
      "\n",
      "3. DC4 6h Ca WT\n",
      "\n",
      "Organism:\tHomo sapiens\n",
      "\n",
      "Source name:\tDC 6h + C. albicans wild type\n",
      "\n",
      "Platform: GPL13667 Series: GSE69723 \n",
      "\n",
      "FTP download: GEO (CEL) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1707nnn/GSM1707896/\n",
      "\n",
      "Sample\t\tAccession: GSM1707896\tID: 301707896\n",
      "\n",
      "\n",
      "\n",
      "4. DC4 6h LPS\n",
      "\n",
      "Organism:\tHomo sapiens\n",
      "\n",
      "Source name:\tDC 6h + LPS\n",
      "\n",
      "Platform: GPL13667 Series: GSE69723 \n",
      "\n",
      "FTP download: GEO (CEL) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1707nnn/GSM1707895/\n",
      "\n",
      "Sample\t\tAccession: GSM1707895\tID: 301707895\n",
      "\n",
      "\n",
      "\n",
      "5. DC4 6h\n",
      "\n",
      "Organism:\tHomo sapiens\n",
      "\n",
      "Source name:\tDC 6h negative control\n",
      "\n",
      "Platform: GPL13667 Series: GSE69723 \n",
      "\n",
      "FTP download: GEO (CEL) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1707nnn/GSM1707894/\n",
      "\n",
      "Sample\t\tAccession: GSM1707894\tID: 301707894\n",
      "\n",
      "\n",
      "\n",
      "6. DC3 6h Af GT\n",
      "\n",
      "Organism:\tHomo sapiens\n",
      "\n",
      "Source name:\tDC 6h + A. fumigatus germ tubes\n",
      "\n",
      "Platform: GPL13667 Series: GSE69723 \n",
      "\n",
      "FTP download: GEO (CEL) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1707nnn/GSM1707893/\n",
      "\n",
      "Sample\t\tAccession: GSM1707893\tID: 301707893\n",
      "\n",
      "\n",
      "\n",
      "7. DC3 6h Ca WT\n",
      "\n",
      "Organism:\tHomo sapiens\n",
      "\n",
      "Source name:\tDC 6h + C. albicans wild type\n",
      "\n",
      "Platform: GPL13667 Series: GSE69723 \n",
      "\n",
      "FTP download: GEO (CEL) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1707nnn/GSM1707892/\n",
      "\n",
      "Sample\t\tAccession: GSM1707892\tID: 301707892\n",
      "\n",
      "\n",
      "\n",
      "8. DC3 6h LPS\n",
      "\n",
      "Organism:\tHomo sapiens\n",
      "\n",
      "Source name:\tDC 6h + LPS\n",
      "\n",
      "Platform: GPL13667 Series: GSE69723 \n",
      "\n",
      "FTP download: GEO (CEL) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1707nnn/GSM1707891/\n",
      "\n",
      "Sample\t\tAccession: GSM1707891\tID: 301707891\n",
      "\n",
      "\n",
      "\n",
      "9. DC3 6h\n",
      "\n",
      "Organism:\tHomo sapiens\n",
      "\n",
      "Source name:\tDC 6h negative control\n",
      "\n",
      "Platform: GPL13667 Series: GSE69723 \n",
      "\n",
      "FTP download: GEO (CEL) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1707nnn/GSM1707890/\n",
      "\n",
      "Sample\t\tAccession: GSM1707890\tID: 301707890\n",
      "\n",
      "\n",
      "\n",
      "10. DC2 6h Af GT\n",
      "\n",
      "Organism:\tHomo sapiens\n",
      "\n",
      "Source name:\tDC 6h + A. fumigatus germ tubes\n",
      "\n",
      "Platform: GPL13667 Series: GSE69723 \n",
      "\n",
      "FTP download: GEO (CEL) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1707nnn/GSM1707889/\n",
      "\n",
      "Sample\t\tAccession: GSM1707889\tID: 301707889\n",
      "\n",
      "Going to download record 51 to 60\n",
      "\n",
      "\n",
      "1. DC2 6h Ca WT\n",
      "\n",
      "Organism:\tHomo sapiens\n",
      "\n",
      "Source name:\tDC 6h + C. albicans wild type\n",
      "\n",
      "Platform: GPL13667 Series: GSE69723 \n",
      "\n",
      "FTP download: GEO (CEL) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1707nnn/GSM1707888/\n",
      "\n",
      "Sample\t\tAccession: GSM1707888\tID: 301707888\n",
      "\n",
      "\n",
      "\n",
      "2. DC2 6h LPS\n",
      "\n",
      "Organism:\tHomo sapiens\n",
      "\n",
      "Source name:\tDC 6h + LPS\n",
      "\n",
      "Platform: GPL13667 Series: GSE69723 \n",
      "\n",
      "FTP download: GEO (CEL) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1707nnn/GSM1707887/\n",
      "\n",
      "Sample\t\tAccession: GSM1707887\tID: 301707887\n",
      "\n",
      "\n",
      "\n",
      "3. DC2 6h\n",
      "\n",
      "Organism:\tHomo sapiens\n",
      "\n",
      "Source name:\tDC 6h negative control\n",
      "\n",
      "Platform: GPL13667 Series: GSE69723 \n",
      "\n",
      "FTP download: GEO (CEL) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1707nnn/GSM1707886/\n",
      "\n",
      "Sample\t\tAccession: GSM1707886\tID: 301707886\n",
      "\n",
      "\n",
      "\n",
      "4. DC1 6h Af GT\n",
      "\n",
      "Organism:\tHomo sapiens\n",
      "\n",
      "Source name:\tDC 6h + A. fumigatus germ tubes\n",
      "\n",
      "Platform: GPL13667 Series: GSE69723 \n",
      "\n",
      "FTP download: GEO (CEL) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1707nnn/GSM1707885/\n",
      "\n",
      "Sample\t\tAccession: GSM1707885\tID: 301707885\n",
      "\n",
      "\n",
      "\n",
      "5. DC1 6h Ca WT\n",
      "\n",
      "Organism:\tHomo sapiens\n",
      "\n",
      "Source name:\tDC 6h + C. albicans wild type\n",
      "\n",
      "Platform: GPL13667 Series: GSE69723 \n",
      "\n",
      "FTP download: GEO (CEL) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1707nnn/GSM1707884/\n",
      "\n",
      "Sample\t\tAccession: GSM1707884\tID: 301707884\n",
      "\n",
      "\n",
      "\n",
      "6. DC1 6h LPS\n",
      "\n",
      "Organism:\tHomo sapiens\n",
      "\n",
      "Source name:\tDC 6h + LPS\n",
      "\n",
      "Platform: GPL13667 Series: GSE69723 \n",
      "\n",
      "FTP download: GEO (CEL) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1707nnn/GSM1707883/\n",
      "\n",
      "Sample\t\tAccession: GSM1707883\tID: 301707883\n",
      "\n",
      "\n",
      "\n",
      "7. DC1 6h\n",
      "\n",
      "Organism:\tHomo sapiens\n",
      "\n",
      "Source name:\tDC 6h negative control\n",
      "\n",
      "Platform: GPL13667 Series: GSE69723 \n",
      "\n",
      "FTP download: GEO (CEL) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1707nnn/GSM1707882/\n",
      "\n",
      "Sample\t\tAccession: GSM1707882\tID: 301707882\n",
      "\n",
      "\n",
      "\n",
      "8. BK/B1\n",
      "\n",
      "Organism:\tSaccharomyces pastorianus\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery B cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery B cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL17932 Series: GSE52379 GSE52380 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1264nnn/GSM1264291/\n",
      "\n",
      "Sample\t\tAccession: GSM1264291\tID: 301264291\n",
      "\n",
      "\n",
      "\n",
      "9. B6/BK\n",
      "\n",
      "Organism:\tSaccharomyces pastorianus\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery B cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery B cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL17932 Series: GSE52379 GSE52380 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1264nnn/GSM1264290/\n",
      "\n",
      "Sample\t\tAccession: GSM1264290\tID: 301264290\n",
      "\n",
      "\n",
      "\n",
      "10. B5/B6\n",
      "\n",
      "Organism:\tSaccharomyces pastorianus\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery B cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery B cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL17932 Series: GSE52379 GSE52380 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1264nnn/GSM1264289/\n",
      "\n",
      "Sample\t\tAccession: GSM1264289\tID: 301264289\n",
      "\n",
      "Going to download record 61 to 70\n",
      "\n",
      "\n",
      "1. B4/B5\n",
      "\n",
      "Organism:\tSaccharomyces pastorianus\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery B cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery B cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL17932 Series: GSE52379 GSE52380 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1264nnn/GSM1264288/\n",
      "\n",
      "Sample\t\tAccession: GSM1264288\tID: 301264288\n",
      "\n",
      "\n",
      "\n",
      "2. B3/B4\n",
      "\n",
      "Organism:\tSaccharomyces pastorianus\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery B cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery B cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL17932 Series: GSE52379 GSE52380 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1264nnn/GSM1264287/\n",
      "\n",
      "Sample\t\tAccession: GSM1264287\tID: 301264287\n",
      "\n",
      "\n",
      "\n",
      "3. B2/B3\n",
      "\n",
      "Organism:\tSaccharomyces pastorianus\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery B cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery B cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL17932 Series: GSE52379 GSE52380 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1264nnn/GSM1264286/\n",
      "\n",
      "Sample\t\tAccession: GSM1264286\tID: 301264286\n",
      "\n",
      "\n",
      "\n",
      "4. B1/B2\n",
      "\n",
      "Organism:\tSaccharomyces pastorianus\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery B cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery B cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL17932 Series: GSE52379 GSE52380 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1264nnn/GSM1264285/\n",
      "\n",
      "Sample\t\tAccession: GSM1264285\tID: 301264285\n",
      "\n",
      "\n",
      "\n",
      "5. AK/A1\n",
      "\n",
      "Organism:\tSaccharomyces pastorianus\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery A cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery A cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL17932 Series: GSE52378 GSE52380 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1264nnn/GSM1264284/\n",
      "\n",
      "Sample\t\tAccession: GSM1264284\tID: 301264284\n",
      "\n",
      "\n",
      "\n",
      "6. A6/AK\n",
      "\n",
      "Organism:\tSaccharomyces pastorianus\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery A cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery A cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL17932 Series: GSE52378 GSE52380 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1264nnn/GSM1264283/\n",
      "\n",
      "Sample\t\tAccession: GSM1264283\tID: 301264283\n",
      "\n",
      "\n",
      "\n",
      "7. A5/A6\n",
      "\n",
      "Organism:\tSaccharomyces pastorianus\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery A cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery A cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL17932 Series: GSE52378 GSE52380 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1264nnn/GSM1264282/\n",
      "\n",
      "Sample\t\tAccession: GSM1264282\tID: 301264282\n",
      "\n",
      "\n",
      "\n",
      "8. A4/A5\n",
      "\n",
      "Organism:\tSaccharomyces pastorianus\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery A cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery A cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL17932 Series: GSE52378 GSE52380 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1264nnn/GSM1264281/\n",
      "\n",
      "Sample\t\tAccession: GSM1264281\tID: 301264281\n",
      "\n",
      "\n",
      "\n",
      "9. A3/A4\n",
      "\n",
      "Organism:\tSaccharomyces pastorianus\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery A cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery A cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL17932 Series: GSE52378 GSE52380 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1264nnn/GSM1264280/\n",
      "\n",
      "Sample\t\tAccession: GSM1264280\tID: 301264280\n",
      "\n",
      "\n",
      "\n",
      "10. A2/A3\n",
      "\n",
      "Organism:\tSaccharomyces pastorianus\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery A cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery A cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL17932 Series: GSE52378 GSE52380 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1264nnn/GSM1264279/\n",
      "\n",
      "Sample\t\tAccession: GSM1264279\tID: 301264279\n",
      "\n",
      "Going to download record 71 to 80\n",
      "\n",
      "\n",
      "1. A1/A2\n",
      "\n",
      "Organism:\tSaccharomyces pastorianus\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery A cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery A cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL17932 Series: GSE52378 GSE52380 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1264nnn/GSM1264278/\n",
      "\n",
      "Sample\t\tAccession: GSM1264278\tID: 301264278\n",
      "\n",
      "\n",
      "\n",
      "2. brewery_A_run_8/brewery_B_run_1\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL2603 Series: GSE45950 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120192/\n",
      "\n",
      "Sample\t\tAccession: GSM1120192\tID: 301120192\n",
      "\n",
      "\n",
      "\n",
      "3. brewery_A_run_1/brewery_B_run_8\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL2603 Series: GSE45950 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120191/\n",
      "\n",
      "Sample\t\tAccession: GSM1120191\tID: 301120191\n",
      "\n",
      "\n",
      "\n",
      "4. brewery_A_run_4/brewery_B_run_1\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL2603 Series: GSE45950 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120190/\n",
      "\n",
      "Sample\t\tAccession: GSM1120190\tID: 301120190\n",
      "\n",
      "\n",
      "\n",
      "5. brewery_A_run_1/brewery_B_run_4\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL2603 Series: GSE45950 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120189/\n",
      "\n",
      "Sample\t\tAccession: GSM1120189\tID: 301120189\n",
      "\n",
      "\n",
      "\n",
      "6. brewery_A_run_5/brewery_B_run_1\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL2603 Series: GSE45950 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120188/\n",
      "\n",
      "Sample\t\tAccession: GSM1120188\tID: 301120188\n",
      "\n",
      "\n",
      "\n",
      "7. brewery_A_run_1/brewery_B_run_5\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL2603 Series: GSE45950 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120187/\n",
      "\n",
      "Sample\t\tAccession: GSM1120187\tID: 301120187\n",
      "\n",
      "\n",
      "\n",
      "8. brewery_A_run_22/brewery_B_run_1\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL2603 Series: GSE45950 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120186/\n",
      "\n",
      "Sample\t\tAccession: GSM1120186\tID: 301120186\n",
      "\n",
      "\n",
      "\n",
      "9. brewery_A_run_1/brewery_B_run_22\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL2603 Series: GSE45950 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120185/\n",
      "\n",
      "Sample\t\tAccession: GSM1120185\tID: 301120185\n",
      "\n",
      "\n",
      "\n",
      "10. SR4-20/SR4-0\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL16996 Series: GSE45947 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120173/\n",
      "\n",
      "Sample\t\tAccession: GSM1120173\tID: 301120173\n",
      "\n",
      "Going to download record 81 to 90\n",
      "\n",
      "\n",
      "1. SR4-19/SR4-20\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL16996 Series: GSE45947 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120172/\n",
      "\n",
      "Sample\t\tAccession: GSM1120172\tID: 301120172\n",
      "\n",
      "\n",
      "\n",
      "2. SR4-18/SR4-19\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL16996 Series: GSE45947 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120171/\n",
      "\n",
      "Sample\t\tAccession: GSM1120171\tID: 301120171\n",
      "\n",
      "\n",
      "\n",
      "3. SR4-17/SR4-18\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL16996 Series: GSE45947 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120170/\n",
      "\n",
      "Sample\t\tAccession: GSM1120170\tID: 301120170\n",
      "\n",
      "\n",
      "\n",
      "4. SR4-16/SR4-17\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL16996 Series: GSE45947 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120169/\n",
      "\n",
      "Sample\t\tAccession: GSM1120169\tID: 301120169\n",
      "\n",
      "\n",
      "\n",
      "5. SR4-15/SR4-16\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL16996 Series: GSE45947 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120168/\n",
      "\n",
      "Sample\t\tAccession: GSM1120168\tID: 301120168\n",
      "\n",
      "\n",
      "\n",
      "6. SR4-13/SR4-15\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL16996 Series: GSE45947 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120167/\n",
      "\n",
      "Sample\t\tAccession: GSM1120167\tID: 301120167\n",
      "\n",
      "\n",
      "\n",
      "7. SR4-12/SR4-13\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL16996 Series: GSE45947 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120166/\n",
      "\n",
      "Sample\t\tAccession: GSM1120166\tID: 301120166\n",
      "\n",
      "\n",
      "\n",
      "8. SR4-11/SR4-12\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL16996 Series: GSE45947 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120165/\n",
      "\n",
      "Sample\t\tAccession: GSM1120165\tID: 301120165\n",
      "\n",
      "\n",
      "\n",
      "9. SR4-10/SR4-11\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL16996 Series: GSE45947 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120164/\n",
      "\n",
      "Sample\t\tAccession: GSM1120164\tID: 301120164\n",
      "\n",
      "\n",
      "\n",
      "10. SR4-9/SR4-10\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL16996 Series: GSE45947 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120163/\n",
      "\n",
      "Sample\t\tAccession: GSM1120163\tID: 301120163\n",
      "\n",
      "Going to download record 91 to 100\n",
      "\n",
      "\n",
      "1. SR4-8/SR4-9\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL16996 Series: GSE45947 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120162/\n",
      "\n",
      "Sample\t\tAccession: GSM1120162\tID: 301120162\n",
      "\n",
      "\n",
      "\n",
      "2. SR4-7/SR4-8\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL16996 Series: GSE45947 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120161/\n",
      "\n",
      "Sample\t\tAccession: GSM1120161\tID: 301120161\n",
      "\n",
      "\n",
      "\n",
      "3. SR4-6/SR4-7\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL16996 Series: GSE45947 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120160/\n",
      "\n",
      "Sample\t\tAccession: GSM1120160\tID: 301120160\n",
      "\n",
      "\n",
      "\n",
      "4. SR4-5/SR4-6\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL16996 Series: GSE45947 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120159/\n",
      "\n",
      "Sample\t\tAccession: GSM1120159\tID: 301120159\n",
      "\n",
      "\n",
      "\n",
      "5. SR4-4/SR4-5\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL16996 Series: GSE45947 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120158/\n",
      "\n",
      "Sample\t\tAccession: GSM1120158\tID: 301120158\n",
      "\n",
      "\n",
      "\n",
      "6. SR4-3/SR4-4\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL16996 Series: GSE45947 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120157/\n",
      "\n",
      "Sample\t\tAccession: GSM1120157\tID: 301120157\n",
      "\n",
      "\n",
      "\n",
      "7. SR4-2/SR4-3\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL16996 Series: GSE45947 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120156/\n",
      "\n",
      "Sample\t\tAccession: GSM1120156\tID: 301120156\n",
      "\n",
      "\n",
      "\n",
      "8. SR4-1/SR4-2\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL16996 Series: GSE45947 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120155/\n",
      "\n",
      "Sample\t\tAccession: GSM1120155\tID: 301120155\n",
      "\n",
      "\n",
      "\n",
      "9. SR4-0/SR4-1\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL16996 Series: GSE45947 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120154/\n",
      "\n",
      "Sample\t\tAccession: GSM1120154\tID: 301120154\n",
      "\n",
      "\n",
      "\n",
      "10. SR3-20/SR3-0\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL16996 Series: GSE45946 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120153/\n",
      "\n",
      "Sample\t\tAccession: GSM1120153\tID: 301120153\n",
      "\n",
      "Going to download record 101 to 110\n",
      "\n",
      "\n",
      "1. SR3-19/SR3-20\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL16996 Series: GSE45946 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120152/\n",
      "\n",
      "Sample\t\tAccession: GSM1120152\tID: 301120152\n",
      "\n",
      "\n",
      "\n",
      "2. SR3-18/SR3-19\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL16996 Series: GSE45946 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120151/\n",
      "\n",
      "Sample\t\tAccession: GSM1120151\tID: 301120151\n",
      "\n",
      "\n",
      "\n",
      "3. SR3-17/SR3-18\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL16996 Series: GSE45946 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120150/\n",
      "\n",
      "Sample\t\tAccession: GSM1120150\tID: 301120150\n",
      "\n",
      "\n",
      "\n",
      "4. SR3-16/SR3-17\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL16996 Series: GSE45946 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120149/\n",
      "\n",
      "Sample\t\tAccession: GSM1120149\tID: 301120149\n",
      "\n",
      "\n",
      "\n",
      "5. SR3-15/SR3-16\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL16996 Series: GSE45946 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120148/\n",
      "\n",
      "Sample\t\tAccession: GSM1120148\tID: 301120148\n",
      "\n",
      "\n",
      "\n",
      "6. SR3-13/SR3-15\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL16996 Series: GSE45946 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120147/\n",
      "\n",
      "Sample\t\tAccession: GSM1120147\tID: 301120147\n",
      "\n",
      "\n",
      "\n",
      "7. SR3-12/SR3-13\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL16996 Series: GSE45946 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120146/\n",
      "\n",
      "Sample\t\tAccession: GSM1120146\tID: 301120146\n",
      "\n",
      "\n",
      "\n",
      "8. SR3-11/SR3-12\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL16996 Series: GSE45946 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120145/\n",
      "\n",
      "Sample\t\tAccession: GSM1120145\tID: 301120145\n",
      "\n",
      "\n",
      "\n",
      "9. SR3-10/SR3-11\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL16996 Series: GSE45946 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120144/\n",
      "\n",
      "Sample\t\tAccession: GSM1120144\tID: 301120144\n",
      "\n",
      "\n",
      "\n",
      "10. SR3-9/SR3-10\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL16996 Series: GSE45946 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120143/\n",
      "\n",
      "Sample\t\tAccession: GSM1120143\tID: 301120143\n",
      "\n",
      "Going to download record 111 to 120\n",
      "\n",
      "\n",
      "1. SR3-8/SR3-9\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL16996 Series: GSE45946 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120142/\n",
      "\n",
      "Sample\t\tAccession: GSM1120142\tID: 301120142\n",
      "\n",
      "\n",
      "\n",
      "2. SR3-7/SR3-8\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL16996 Series: GSE45946 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120141/\n",
      "\n",
      "Sample\t\tAccession: GSM1120141\tID: 301120141\n",
      "\n",
      "\n",
      "\n",
      "3. SR3-6/SR3-7\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL16996 Series: GSE45946 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120140/\n",
      "\n",
      "Sample\t\tAccession: GSM1120140\tID: 301120140\n",
      "\n",
      "\n",
      "\n",
      "4. SR3-5/SR3-6\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL16996 Series: GSE45946 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120139/\n",
      "\n",
      "Sample\t\tAccession: GSM1120139\tID: 301120139\n",
      "\n",
      "\n",
      "\n",
      "5. SR3-4/SR3-5\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL16996 Series: GSE45946 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120138/\n",
      "\n",
      "Sample\t\tAccession: GSM1120138\tID: 301120138\n",
      "\n",
      "\n",
      "\n",
      "6. SR3-3/SR3-4\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL16996 Series: GSE45946 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120137/\n",
      "\n",
      "Sample\t\tAccession: GSM1120137\tID: 301120137\n",
      "\n",
      "\n",
      "\n",
      "7. SR3-2/SR3-3\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL16996 Series: GSE45946 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120136/\n",
      "\n",
      "Sample\t\tAccession: GSM1120136\tID: 301120136\n",
      "\n",
      "\n",
      "\n",
      "8. SR3-1/SR3-2\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL16996 Series: GSE45946 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120135/\n",
      "\n",
      "Sample\t\tAccession: GSM1120135\tID: 301120135\n",
      "\n",
      "\n",
      "\n",
      "9. SR3-0/SR3-1\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL16996 Series: GSE45946 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120134/\n",
      "\n",
      "Sample\t\tAccession: GSM1120134\tID: 301120134\n",
      "\n",
      "\n",
      "\n",
      "10. SR2-17/SR2-0\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL16996 Series: GSE45944 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120115/\n",
      "\n",
      "Sample\t\tAccession: GSM1120115\tID: 301120115\n",
      "\n",
      "Going to download record 121 to 130\n",
      "\n",
      "\n",
      "1. SR2-16/SR2-17\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL16996 Series: GSE45944 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120114/\n",
      "\n",
      "Sample\t\tAccession: GSM1120114\tID: 301120114\n",
      "\n",
      "\n",
      "\n",
      "2. SR2-15/SR2-16\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL16996 Series: GSE45944 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120113/\n",
      "\n",
      "Sample\t\tAccession: GSM1120113\tID: 301120113\n",
      "\n",
      "\n",
      "\n",
      "3. SR2-14/SR2-15\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL16996 Series: GSE45944 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120112/\n",
      "\n",
      "Sample\t\tAccession: GSM1120112\tID: 301120112\n",
      "\n",
      "\n",
      "\n",
      "4. SR2-13/SR2-14\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL16996 Series: GSE45944 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120111/\n",
      "\n",
      "Sample\t\tAccession: GSM1120111\tID: 301120111\n",
      "\n",
      "\n",
      "\n",
      "5. SR2-12/SR2-13\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL16996 Series: GSE45944 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120110/\n",
      "\n",
      "Sample\t\tAccession: GSM1120110\tID: 301120110\n",
      "\n",
      "\n",
      "\n",
      "6. SR2-11/SR2-12\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL16996 Series: GSE45944 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120109/\n",
      "\n",
      "Sample\t\tAccession: GSM1120109\tID: 301120109\n",
      "\n",
      "\n",
      "\n",
      "7. SR2-9/SR2-11\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL16996 Series: GSE45944 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120108/\n",
      "\n",
      "Sample\t\tAccession: GSM1120108\tID: 301120108\n",
      "\n",
      "\n",
      "\n",
      "8. SR2-8/SR2-9\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL16996 Series: GSE45944 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120107/\n",
      "\n",
      "Sample\t\tAccession: GSM1120107\tID: 301120107\n",
      "\n",
      "\n",
      "\n",
      "9. SR2-7/SR2-8\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL16996 Series: GSE45944 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120106/\n",
      "\n",
      "Sample\t\tAccession: GSM1120106\tID: 301120106\n",
      "\n",
      "\n",
      "\n",
      "10. SR2-6/SR2-7\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL16996 Series: GSE45944 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120105/\n",
      "\n",
      "Sample\t\tAccession: GSM1120105\tID: 301120105\n",
      "\n",
      "Going to download record 131 to 140\n",
      "\n",
      "\n",
      "1. SR2-5/SR2-6\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL16996 Series: GSE45944 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120104/\n",
      "\n",
      "Sample\t\tAccession: GSM1120104\tID: 301120104\n",
      "\n",
      "\n",
      "\n",
      "2. SR2-4/SR2-5\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL16996 Series: GSE45944 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120103/\n",
      "\n",
      "Sample\t\tAccession: GSM1120103\tID: 301120103\n",
      "\n",
      "\n",
      "\n",
      "3. SR2-3/SR2-4\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL16996 Series: GSE45944 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120102/\n",
      "\n",
      "Sample\t\tAccession: GSM1120102\tID: 301120102\n",
      "\n",
      "\n",
      "\n",
      "4. SR2-2/SR2-3\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL16996 Series: GSE45944 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120101/\n",
      "\n",
      "Sample\t\tAccession: GSM1120101\tID: 301120101\n",
      "\n",
      "\n",
      "\n",
      "5. SR2-1/SR2-2\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL16996 Series: GSE45944 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120100/\n",
      "\n",
      "Sample\t\tAccession: GSM1120100\tID: 301120100\n",
      "\n",
      "\n",
      "\n",
      "6. SR2-0/SR2-1\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL16996 Series: GSE45944 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120099/\n",
      "\n",
      "Sample\t\tAccession: GSM1120099\tID: 301120099\n",
      "\n",
      "\n",
      "\n",
      "7. SR1-18/SR1-1\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL16996 Series: GSE45943 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120098/\n",
      "\n",
      "Sample\t\tAccession: GSM1120098\tID: 301120098\n",
      "\n",
      "\n",
      "\n",
      "8. SR1-17/SR1-18\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL16996 Series: GSE45943 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120097/\n",
      "\n",
      "Sample\t\tAccession: GSM1120097\tID: 301120097\n",
      "\n",
      "\n",
      "\n",
      "9. SR1-16/SR1-17\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL16996 Series: GSE45943 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120096/\n",
      "\n",
      "Sample\t\tAccession: GSM1120096\tID: 301120096\n",
      "\n",
      "\n",
      "\n",
      "10. SR1-15/SR1-16\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL16996 Series: GSE45943 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120095/\n",
      "\n",
      "Sample\t\tAccession: GSM1120095\tID: 301120095\n",
      "\n",
      "Going to download record 141 to 150\n",
      "\n",
      "\n",
      "1. SR1-14/SR1-15\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL16996 Series: GSE45943 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120094/\n",
      "\n",
      "Sample\t\tAccession: GSM1120094\tID: 301120094\n",
      "\n",
      "\n",
      "\n",
      "2. SR1-13/SR1-14\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL16996 Series: GSE45943 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120093/\n",
      "\n",
      "Sample\t\tAccession: GSM1120093\tID: 301120093\n",
      "\n",
      "\n",
      "\n",
      "3. SR1-12/SR1-13\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL16996 Series: GSE45943 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120092/\n",
      "\n",
      "Sample\t\tAccession: GSM1120092\tID: 301120092\n",
      "\n",
      "\n",
      "\n",
      "4. SR1-11/SR1-12\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL16996 Series: GSE45943 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120091/\n",
      "\n",
      "Sample\t\tAccession: GSM1120091\tID: 301120091\n",
      "\n",
      "\n",
      "\n",
      "5. SR1-10/SR1-11\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL16996 Series: GSE45943 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120090/\n",
      "\n",
      "Sample\t\tAccession: GSM1120090\tID: 301120090\n",
      "\n",
      "\n",
      "\n",
      "6. SR1-9/SR1-10\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL16996 Series: GSE45943 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120089/\n",
      "\n",
      "Sample\t\tAccession: GSM1120089\tID: 301120089\n",
      "\n",
      "\n",
      "\n",
      "7. SR1-8/SR1-9\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL16996 Series: GSE45943 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120088/\n",
      "\n",
      "Sample\t\tAccession: GSM1120088\tID: 301120088\n",
      "\n",
      "\n",
      "\n",
      "8. SR1-7/SR1-8\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL16996 Series: GSE45943 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120087/\n",
      "\n",
      "Sample\t\tAccession: GSM1120087\tID: 301120087\n",
      "\n",
      "\n",
      "\n",
      "9. SR1-6/SR1-7\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL16996 Series: GSE45943 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120086/\n",
      "\n",
      "Sample\t\tAccession: GSM1120086\tID: 301120086\n",
      "\n",
      "\n",
      "\n",
      "10. SR1-5/SR1-6\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL16996 Series: GSE45943 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120085/\n",
      "\n",
      "Sample\t\tAccession: GSM1120085\tID: 301120085\n",
      "\n",
      "Going to download record 151 to 160\n",
      "\n",
      "\n",
      "1. SR1-4/SR1-5\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL16996 Series: GSE45943 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120084/\n",
      "\n",
      "Sample\t\tAccession: GSM1120084\tID: 301120084\n",
      "\n",
      "\n",
      "\n",
      "2. SR1-3/SR1-4\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL16996 Series: GSE45943 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120083/\n",
      "\n",
      "Sample\t\tAccession: GSM1120083\tID: 301120083\n",
      "\n",
      "\n",
      "\n",
      "3. SR1-2/SR1-3\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL16996 Series: GSE45943 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120082/\n",
      "\n",
      "Sample\t\tAccession: GSM1120082\tID: 301120082\n",
      "\n",
      "\n",
      "\n",
      "4. SR1-1/SR1-2\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tyeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 1) yeast cells harvested by the end of fermentation from brewery cylindro-conical vessel (channel 2)\n",
      "\n",
      "Platform: GPL16996 Series: GSE45943 GSE45951 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1120nnn/GSM1120081/\n",
      "\n",
      "Sample\t\tAccession: GSM1120081\tID: 301120081\n",
      "\n",
      "\n",
      "\n",
      "5. CBS1502\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tSbay CBS7001 + Scer S288C  genomic DNA sonicated, 400 ng ea (channel 1) CBS1502 genomic DNA 350 ng HaeIII cut (channel 2)\n",
      "\n",
      "Platform: GPL7077 Series: GSE35528 \n",
      "\n",
      "Sample\t\tAccession: GSM870219\tID: 300870219\n",
      "\n",
      "\n",
      "\n",
      "6. Sc-ale vs. Sc DNA\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tDBY7286 genomic DNA (channel 1) ale (S. cerevisiae) genomic DNA (channel 2)\n",
      "\n",
      "Platform: GPL55 Series: GSE23055 \n",
      "\n",
      "Sample\t\tAccession: GSM568798\tID: 300568798\n",
      "\n",
      "\n",
      "\n",
      "7. Sc-ale vs. Sc DNA\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tDBY7286 genomic DNA (channel 1) ale (S. cerevisiae) genomic DNA (channel 2)\n",
      "\n",
      "Platform: GPL7756 Series: GSE13875 \n",
      "\n",
      "Sample\t\tAccession: GSM349159\tID: 300349159\n",
      "\n",
      "\n",
      "\n",
      "8. GSY508-DyeSwap\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tGSY508 genomic DNA 350 ng HaeIII cut (channel 1) Sbay CBS7001 + Scer S288C  genomic DNA sonicated, 400 ng ea (channel 2)\n",
      "\n",
      "Platform: GPL7077 Series: GSE12177 \n",
      "\n",
      "Sample\t\tAccession: GSM406527\tID: 300406527\n",
      "\n",
      "\n",
      "\n",
      "9. GSY504-DyeSwap\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tGSY504 genomic DNA 350 ng HaeIII cut (channel 1) Sbay CBS7001 + Scer S288C genomic DNA, sonicated, 400 ng each (channel 2)\n",
      "\n",
      "Platform: GPL7077 Series: GSE12177 \n",
      "\n",
      "Sample\t\tAccession: GSM406526\tID: 300406526\n",
      "\n",
      "\n",
      "\n",
      "10. GSY503-DyeSwap\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tGSY503 genomic DNA 350 ng HaeIII cut (channel 1) Sbay CBS7001 + Scer S288C  genomic DNA sonicated, 400 ng ea (channel 2)\n",
      "\n",
      "Platform: GPL7077 Series: GSE12177 \n",
      "\n",
      "Sample\t\tAccession: GSM406525\tID: 300406525\n",
      "\n",
      "Going to download record 161 to 170\n",
      "\n",
      "\n",
      "1. GSY509-DyeSwap\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tGSY509 genomic DNA 350 ng HaeIII cut (channel 1) Sbay CBS7001 + Scer S288C  genomic DNA sonicated, 400 ng ea (channel 2)\n",
      "\n",
      "Platform: GPL7077 Series: GSE12177 \n",
      "\n",
      "Sample\t\tAccession: GSM406524\tID: 300406524\n",
      "\n",
      "\n",
      "\n",
      "2. GSY501-DyeSwap\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tGSY501 genomic DNA 350 ng HaeIII cut (channel 1) Sbay CBS7001 + Scer S288C  genomic DNA sonicated, 400 ng ea (channel 2)\n",
      "\n",
      "Platform: GPL7077 Series: GSE12177 \n",
      "\n",
      "Sample\t\tAccession: GSM406523\tID: 300406523\n",
      "\n",
      "\n",
      "\n",
      "3. GSY504-CBS1484\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tScer 350ng + Sbay 350ng sonicated genomic DNA (channel 1) GSY504 genomic DNA, 350 ng, HaeIII cut (channel 2)\n",
      "\n",
      "Platform: GPL7077 Series: GSE12177 \n",
      "\n",
      "Sample\t\tAccession: GSM306622\tID: 300306622\n",
      "\n",
      "\n",
      "\n",
      "4. GSY133\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tScer 350ng + Sbay 350ng sonicated genomic DNA (channel 1) GSY133 genomic DNA, 350 ng, HaeIII cut (channel 2)\n",
      "\n",
      "Platform: GPL7077 Series: GSE12177 \n",
      "\n",
      "Sample\t\tAccession: GSM306621\tID: 300306621\n",
      "\n",
      "\n",
      "\n",
      "5. GSY516\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tSbay CBS7001 + Scer S288C  genomic DNA sonicated, 350 ng ea (channel 1) GSY516 genomic DNA 350 ng HaeIII cut (channel 2)\n",
      "\n",
      "Platform: GPL7077 Series: GSE12177 \n",
      "\n",
      "Sample\t\tAccession: GSM306620\tID: 300306620\n",
      "\n",
      "\n",
      "\n",
      "6. GSY138\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tScer 350ng + Sbay 350ng sonicated genomic DNA (channel 1) GSY138 genomic DNA, 350 ng, HaeIII cut (channel 2)\n",
      "\n",
      "Platform: GPL7077 Series: GSE12177 \n",
      "\n",
      "Sample\t\tAccession: GSM306619\tID: 300306619\n",
      "\n",
      "\n",
      "\n",
      "7. GSY131\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tScer 350ng + Sbay 350ng sonicated genomic DNA (channel 1) GSY131 genomic DNA, 350 ng, HaeIII cut (channel 2)\n",
      "\n",
      "Platform: GPL7077 Series: GSE12177 \n",
      "\n",
      "Sample\t\tAccession: GSM306618\tID: 300306618\n",
      "\n",
      "\n",
      "\n",
      "8. Self-HaeIII-Sbay-Scer\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tScer 400ng + Sbay 300ng sonicated genomic DNA (channel 1) Sbay Zymo prep 300 ng, HaeIII cut, Scer Zymo prep 100 ng, HaeIII cut plus 300ng uncut (channel 2)\n",
      "\n",
      "Platform: GPL7077 Series: GSE12177 GSE35528 \n",
      "\n",
      "Sample\t\tAccession: GSM306617\tID: 300306617\n",
      "\n",
      "\n",
      "\n",
      "9. GSY508-CBS2156\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tScer 400ng + Sbay 300ng sonicated genomic DNA (channel 1) GSY508 genomic DNA, 350 ng, HaeIII cut (channel 2)\n",
      "\n",
      "Platform: GPL7077 Series: GSE12177 \n",
      "\n",
      "Sample\t\tAccession: GSM306616\tID: 300306616\n",
      "\n",
      "\n",
      "\n",
      "10. GSY501-CBS1174\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tScer 400ng + Sbay 300ng sonicated genomic DNA (channel 1) GSY501 genomic DNA, 350 ng, HaeIII cut (channel 2)\n",
      "\n",
      "Platform: GPL7077 Series: GSE12177 \n",
      "\n",
      "Sample\t\tAccession: GSM306615\tID: 300306615\n",
      "\n",
      "Going to download record 171 to 180\n",
      "\n",
      "\n",
      "1. GSY137\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tScer 350ng + Sbay 350ng sonicated genomic DNA (channel 1) GSY137 genomic DNA, 350 ng, HaeIII cut (channel 2)\n",
      "\n",
      "Platform: GPL7077 Series: GSE12177 \n",
      "\n",
      "Sample\t\tAccession: GSM306614\tID: 300306614\n",
      "\n",
      "\n",
      "\n",
      "2. GSY132\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tScer 350ng + Sbay 350ng sonicated genomic DNA (channel 1) GSY132 genomic DNA, 350 ng, HaeIII cut (channel 2)\n",
      "\n",
      "Platform: GPL7077 Series: GSE12177 \n",
      "\n",
      "Sample\t\tAccession: GSM306613\tID: 300306613\n",
      "\n",
      "\n",
      "\n",
      "3. GSY503-CBS1483\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tScer  350ng + Sbay 350ng sonicated genomic DNA (channel 1) GSY503 genomic DNA, 350 ng, HaeIII cut (channel 2)\n",
      "\n",
      "Platform: GPL7077 Series: GSE12177 \n",
      "\n",
      "Sample\t\tAccession: GSM306612\tID: 300306612\n",
      "\n",
      "\n",
      "\n",
      "4. GSY515\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tSbay CBS7001 + Scer S288C  genomic DNA sonicated, 350 ng ea (channel 1) GSY515 genomic DNA 350 ng HaeIII cut (channel 2)\n",
      "\n",
      "Platform: GPL7077 Series: GSE12177 \n",
      "\n",
      "Sample\t\tAccession: GSM306611\tID: 300306611\n",
      "\n",
      "\n",
      "\n",
      "5. GSY509\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tSbay CBS7001 + Scer S288C  genomic DNA sonicated, 350 ng ea (channel 1) GSY509 genomic DNA 350 ng HaeIII cut (channel 2)\n",
      "\n",
      "Platform: GPL7077 Series: GSE12177 \n",
      "\n",
      "Sample\t\tAccession: GSM306610\tID: 300306610\n",
      "\n",
      "\n",
      "\n",
      "6. GSY136-22Aug06-65Wash\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tScer + Sbay genomic each DNA 350 ng sonicated (channel 1) GSY136 genomic DNA 350 ng HaeIII cut (channel 2)\n",
      "\n",
      "Platform: GPL7077 Series: GSE12177 \n",
      "\n",
      "Sample\t\tAccession: GSM306609\tID: 300306609\n",
      "\n",
      "\n",
      "\n",
      "7. GSY129-22Aug06-65Wash\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tScer + Sbay genomic each DNA 350 ng sonicated (channel 1) GSY129 genomic DNA 350 ng HaeIII cut (channel 2)\n",
      "\n",
      "Platform: GPL7077 Series: GSE12177 \n",
      "\n",
      "Sample\t\tAccession: GSM306608\tID: 300306608\n",
      "\n",
      "\n",
      "\n",
      "8. GSY134-22Aug06-65Wash\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tScer + Sbay genomic each DNA 350 ng sonicated (channel 1) GSY134 genomic DNA 350 ng HaeIII cut (channel 2)\n",
      "\n",
      "Platform: GPL7077 Series: GSE12177 \n",
      "\n",
      "Sample\t\tAccession: GSM306607\tID: 300306607\n",
      "\n",
      "\n",
      "\n",
      "9. GSY135\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tScer + Sbay genomic DNA 350 ng each sonicated (channel 1) GSY135 genomic DNA 350 ng HaeIII cut (channel 2)\n",
      "\n",
      "Platform: GPL7077 Series: GSE12177 \n",
      "\n",
      "Sample\t\tAccession: GSM306606\tID: 300306606\n",
      "\n",
      "\n",
      "\n",
      "10. Sbay + Scer HaeIII 65Wash\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tScer + Sbay genomic DNA 350 ng each sonicated (channel 1) Sbay + Scer genomic DNA 350 ng HaeIII cut (channel 2)\n",
      "\n",
      "Platform: GPL7077 Series: GSE12177 GSE35528 \n",
      "\n",
      "Sample\t\tAccession: GSM306603\tID: 300306603\n",
      "\n",
      "Going to download record 181 to 190\n",
      "\n",
      "\n",
      "1. Self-HaeIII vs Self-Sonicated\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tSbay + Scer genomic DNA 350 ng each sonicated (channel 1) Sbay + Scer genomic DNA  350 ng each HaeIII cut (channel 2)\n",
      "\n",
      "Platform: GPL7077 Series: GSE12177 GSE35528 \n",
      "\n",
      "Sample\t\tAccession: GSM306602\tID: 300306602\n",
      "\n",
      "\n",
      "\n",
      "2. GSY139-22Aug06-65Wash\n",
      "\n",
      "Organism:\tSaccharomyces cerevisiae\n",
      "\n",
      "Source name:\tScer + Sbay genomic each DNA 350 ng sonicated (channel 1) GSY139 genomic DNA 350 ng HaeIII cut (channel 2)\n",
      "\n",
      "Platform: GPL7077 Series: GSE12177 \n",
      "\n",
      "Sample\t\tAccession: GSM306601\tID: 300306601\n",
      "\n",
      "\n",
      "\n",
      "3. Megasphaera cerevisiae/Pectinatus cerevisiiphilus mix detected by microarray\n",
      "\n",
      "Organism:\tMegasphaera cerevisiae; Pectinatus cerevisiiphilus\n",
      "\n",
      "Source name:\tBeer spoilage bacteria\n",
      "\n",
      "Platform: GPL5157 Series: GSE7840 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM190nnn/GSM190244/\n",
      "\n",
      "Sample\t\tAccession: GSM190244\tID: 300190244\n",
      "\n",
      "\n",
      "\n",
      "4. Pectinatus cerevisiiphilus detected by microarray\n",
      "\n",
      "Organism:\tPectinatus cerevisiiphilus\n",
      "\n",
      "Source name:\tBeer spoilage bacteria\n",
      "\n",
      "Platform: GPL5157 Series: GSE7840 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM190nnn/GSM190242/\n",
      "\n",
      "Sample\t\tAccession: GSM190242\tID: 300190242\n",
      "\n",
      "\n",
      "\n",
      "5. Megasphaera cerevisiae detected by microarray\n",
      "\n",
      "Organism:\tMegasphaera cerevisiae\n",
      "\n",
      "Source name:\tBeer spoilage bacteria\n",
      "\n",
      "Platform: GPL5157 Series: GSE7840 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM190nnn/GSM190240/\n",
      "\n",
      "Sample\t\tAccession: GSM190240\tID: 300190240\n",
      "\n",
      "\n",
      "\n",
      "6. Pediococcus inopinatus and Pectinatus cerevisiiphilus detected by microarray\n",
      "\n",
      "Organism:\tPediococcus inopinatus; Pectinatus cerevisiiphilus\n",
      "\n",
      "Source name:\tBeer spoilage bacteria\n",
      "\n",
      "Platform: GPL5158 Series: GSE7840 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM190nnn/GSM190249/\n",
      "\n",
      "Sample\t\tAccession: GSM190249\tID: 300190249\n",
      "\n",
      "\n",
      "\n",
      "7. Lactobacillus brevis and Pectinatus frisingensis detected by microarray\n",
      "\n",
      "Organism:\tPectinatus frisingensis; Lactobacillus brevis\n",
      "\n",
      "Source name:\tBeer spoilage bacteria\n",
      "\n",
      "Platform: GPL5158 Series: GSE7840 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM190nnn/GSM190248/\n",
      "\n",
      "Sample\t\tAccession: GSM190248\tID: 300190248\n",
      "\n",
      "\n",
      "\n",
      "8. Pectinatus frisingensis detected by microarray\n",
      "\n",
      "Organism:\tPectinatus frisingensis\n",
      "\n",
      "Source name:\tBeer spoilage bacteria\n",
      "\n",
      "Platform: GPL5158 Series: GSE7840 \n",
      "\n",
      "FTP download: GEO (GPR) ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM190nnn/GSM190247/\n",
      "\n",
      "Sample\t\tAccession: GSM190247\tID: 300190247\n",
      "\n",
      "\n",
      "\n",
      "9. Lager beer fermentation 264 hours\n",
      "\n",
      "Organism:\tSaccharomyces pastorianus\n",
      "\n",
      "Source name:\t264 hours\n",
      "\n",
      "Platform: GPL205 Series: GSE79 \n",
      "\n",
      "Sample\t\tAccession: GSM2381\tID: 300002381\n",
      "\n",
      "\n",
      "\n",
      "10. Lager beer fermentation 240 hours\n",
      "\n",
      "Organism:\tSaccharomyces pastorianus\n",
      "\n",
      "Source name:\t240 hours\n",
      "\n",
      "Platform: GPL205 Series: GSE79 \n",
      "\n",
      "Sample\t\tAccession: GSM2380\tID: 300002380\n",
      "\n",
      "Going to download record 191 to 200\n",
      "\n",
      "\n",
      "1. Lager beer fermentation 192 hours\n",
      "\n",
      "Organism:\tSaccharomyces pastorianus\n",
      "\n",
      "Source name:\t192 hours\n",
      "\n",
      "Platform: GPL205 Series: GSE79 \n",
      "\n",
      "Sample\t\tAccession: GSM2379\tID: 300002379\n",
      "\n",
      "\n",
      "\n",
      "2. Lager beer fermentation 168 hours\n",
      "\n",
      "Organism:\tSaccharomyces pastorianus\n",
      "\n",
      "Source name:\t168 hours\n",
      "\n",
      "Platform: GPL205 Series: GSE79 \n",
      "\n",
      "Sample\t\tAccession: GSM2378\tID: 300002378\n",
      "\n",
      "\n",
      "\n",
      "3. Lager beer fermentation 144 hours\n",
      "\n",
      "Organism:\tSaccharomyces pastorianus\n",
      "\n",
      "Source name:\t144 hours\n",
      "\n",
      "Platform: GPL205 Series: GSE79 \n",
      "\n",
      "Sample\t\tAccession: GSM2377\tID: 300002377\n",
      "\n",
      "\n",
      "\n",
      "4. Lager beer fermentation 120 hours\n",
      "\n",
      "Organism:\tSaccharomyces pastorianus\n",
      "\n",
      "Source name:\t120 hours\n",
      "\n",
      "Platform: GPL205 Series: GSE79 \n",
      "\n",
      "Sample\t\tAccession: GSM2376\tID: 300002376\n",
      "\n",
      "\n",
      "\n",
      "5. Lager beer fermentation 96 hours\n",
      "\n",
      "Organism:\tSaccharomyces pastorianus\n",
      "\n",
      "Source name:\t96 hours\n",
      "\n",
      "Platform: GPL205 Series: GSE79 \n",
      "\n",
      "Sample\t\tAccession: GSM2375\tID: 300002375\n",
      "\n",
      "\n",
      "\n",
      "6. Lager beer fermentation 72 hours\n",
      "\n",
      "Organism:\tSaccharomyces pastorianus\n",
      "\n",
      "Source name:\t72 hours\n",
      "\n",
      "Platform: GPL205 Series: GSE79 \n",
      "\n",
      "Sample\t\tAccession: GSM2374\tID: 300002374\n",
      "\n",
      "\n",
      "\n",
      "7. Lager beer fermentation 48 hours\n",
      "\n",
      "Organism:\tSaccharomyces pastorianus\n",
      "\n",
      "Source name:\t48 hours\n",
      "\n",
      "Platform: GPL205 Series: GSE79 \n",
      "\n",
      "Sample\t\tAccession: GSM2373\tID: 300002373\n",
      "\n",
      "\n",
      "\n",
      "8. Lager beer fermentation 24 hours\n",
      "\n",
      "Organism:\tSaccharomyces pastorianus\n",
      "\n",
      "Source name:\t24 hours\n",
      "\n",
      "Platform: GPL205 Series: GSE79 \n",
      "\n",
      "Sample\t\tAccession: GSM2372\tID: 300002372\n",
      "\n",
      "\n",
      "\n",
      "9. Lager beer fermentation 1 hours\n",
      "\n",
      "Organism:\tSaccharomyces pastorianus\n",
      "\n",
      "Source name:\t1 hours\n",
      "\n",
      "Platform: GPL205 Series: GSE79 \n",
      "\n",
      "Sample\t\tAccession: GSM2371\tID: 300002371\n",
      "\n",
      "\n",
      "\n",
      "10. Lager beer fermentation 0 hours\n",
      "\n",
      "Organism:\tSaccharomyces pastorianus\n",
      "\n",
      "Source name:\t0 hours\n",
      "\n",
      "Platform: GPL205 Series: GSE79 \n",
      "\n",
      "Sample\t\tAccession: GSM2370\tID: 300002370\n",
      "\n",
      "handle is type <class '_io.TextIOWrapper'>\n"
     ]
    }
   ],
   "source": [
    "from Bio import Entrez\n",
    "import time\n",
    "\n",
    "try:\n",
    "    from urllib.error import HTTPError  # for Python 3\n",
    "except ImportError:\n",
    "    from urllib2 import HTTPError  # for Python 2\n",
    "\n",
    "Entrez.email = \"astuart@ramapo.edu\"\n",
    "\n",
    "handle = Entrez.esearch(db=\"gds\", term=\"Saccharomyces[kywd] AND beer[kywd]\", RetMax='10', usehistory='y')\n",
    "search_results = Entrez.read(handle)\n",
    "\n",
    "print(\"The WebEnv is {}\".format(search_results[\"WebEnv\"]))\n",
    "print(\"The QueryKey is {}\".format(search_results[\"QueryKey\"]))\n",
    "\n",
    "print(\"Data Type of search_results \",type(search_results))\n",
    "print(search_results.keys())\n",
    "print(\"Results found =\",search_results[\"Count\"])\n",
    "count = int(search_results[\"Count\"])\n",
    "print(\"Found %i results\" % count)\n",
    "print(\"Query Translation =\",search_results[\"QueryTranslation\"])\n",
    "idlist = search_results[\"IdList\"]\n",
    "print(search_results[\"IdList\"])\n",
    "\n",
    "\n",
    "batch_size = 10\n",
    "\n",
    "for start in range(0,count,batch_size):\n",
    "    end = min(count, start+batch_size)\n",
    "    print(\"Going to download record %i to %i\" % (start+1, end))\n",
    "\n",
    "    fetch_handle = Entrez.efetch(db=\"gds\", retmode=\"text\",\n",
    "                                         retstart=start,\n",
    "                                         retmax=10,\n",
    "                                         #retmax=batch_size,\n",
    "                                         webenv=search_results[\"WebEnv\"],\n",
    "                                         query_key=search_results[\"QueryKey\"])\n",
    "            \n",
    "    for line in fetch_handle:\n",
    "              print(line)\n",
    "            \n",
    "    #fetch_handle.close()\n",
    "    \n",
    "print(\"handle is type\",type(fetch_handle))"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.6"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
